These 10 CAs from the occupied area of the Republic of China should not be trusted.
--- a/mozilla/certdata.txt
+++ b/mozilla/certdata.txt
@@ -10594,174 +10594,6 @@ CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
 CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
 CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
 
-#
-# Certificate "CFCA EV ROOT"
-#
-# Issuer: CN=CFCA EV ROOT,O=China Financial Certification Authority,C=CN
-# Serial Number: 407555286 (0x184accd6)
-# Subject: CN=CFCA EV ROOT,O=China Financial Certification Authority,C=CN
-# Not Valid Before: Wed Aug 08 03:07:01 2012
-# Not Valid After : Mon Dec 31 03:07:01 2029
-# Fingerprint (SHA-256): 5C:C3:D7:8E:4E:1D:5E:45:54:7A:04:E6:87:3E:64:F9:0C:F9:53:6D:1C:CC:2E:F8:00:F3:55:C4:C5:FD:70:FD
-# Fingerprint (SHA1): E2:B8:29:4B:55:84:AB:6B:58:C2:90:46:6C:AC:3F:B8:39:8F:84:83
-CKA_CLASS CK_OBJECT_CLASS CKO_CERTIFICATE
-CKA_TOKEN CK_BBOOL CK_TRUE
-CKA_PRIVATE CK_BBOOL CK_FALSE
-CKA_MODIFIABLE CK_BBOOL CK_FALSE
-CKA_LABEL UTF8 "CFCA EV ROOT"
-CKA_CERTIFICATE_TYPE CK_CERTIFICATE_TYPE CKC_X_509
-CKA_SUBJECT MULTILINE_OCTAL
-\060\126\061\013\060\011\006\003\125\004\006\023\002\103\116\061
-\060\060\056\006\003\125\004\012\014\047\103\150\151\156\141\040
-\106\151\156\141\156\143\151\141\154\040\103\145\162\164\151\146
-\151\143\141\164\151\157\156\040\101\165\164\150\157\162\151\164
-\171\061\025\060\023\006\003\125\004\003\014\014\103\106\103\101
-\040\105\126\040\122\117\117\124
-END
-CKA_ID UTF8 "0"
-CKA_ISSUER MULTILINE_OCTAL
-\060\126\061\013\060\011\006\003\125\004\006\023\002\103\116\061
-\060\060\056\006\003\125\004\012\014\047\103\150\151\156\141\040
-\106\151\156\141\156\143\151\141\154\040\103\145\162\164\151\146
-\151\143\141\164\151\157\156\040\101\165\164\150\157\162\151\164
-\171\061\025\060\023\006\003\125\004\003\014\014\103\106\103\101
-\040\105\126\040\122\117\117\124
-END
-CKA_SERIAL_NUMBER MULTILINE_OCTAL
-\002\004\030\112\314\326
-END
-CKA_VALUE MULTILINE_OCTAL
-\060\202\005\215\060\202\003\165\240\003\002\001\002\002\004\030
-\112\314\326\060\015\006\011\052\206\110\206\367\015\001\001\013
-\005\000\060\126\061\013\060\011\006\003\125\004\006\023\002\103
-\116\061\060\060\056\006\003\125\004\012\014\047\103\150\151\156
-\141\040\106\151\156\141\156\143\151\141\154\040\103\145\162\164
-\151\146\151\143\141\164\151\157\156\040\101\165\164\150\157\162
-\151\164\171\061\025\060\023\006\003\125\004\003\014\014\103\106
-\103\101\040\105\126\040\122\117\117\124\060\036\027\015\061\062
-\060\070\060\070\060\063\060\067\060\061\132\027\015\062\071\061
-\062\063\061\060\063\060\067\060\061\132\060\126\061\013\060\011
-\006\003\125\004\006\023\002\103\116\061\060\060\056\006\003\125
-\004\012\014\047\103\150\151\156\141\040\106\151\156\141\156\143
-\151\141\154\040\103\145\162\164\151\146\151\143\141\164\151\157
-\156\040\101\165\164\150\157\162\151\164\171\061\025\060\023\006
-\003\125\004\003\014\014\103\106\103\101\040\105\126\040\122\117
-\117\124\060\202\002\042\060\015\006\011\052\206\110\206\367\015
-\001\001\001\005\000\003\202\002\017\000\060\202\002\012\002\202
-\002\001\000\327\135\153\315\020\077\037\005\131\325\005\115\067
-\261\016\354\230\053\216\025\035\372\223\113\027\202\041\161\020
-\122\327\121\144\160\026\302\125\151\115\216\025\155\237\277\014
-\033\302\340\243\147\326\014\254\317\042\256\257\167\124\052\113
-\114\212\123\122\172\303\356\056\336\263\161\045\301\351\135\075
-\356\241\057\243\367\052\074\311\043\035\152\253\035\241\247\361
-\363\354\240\325\104\317\025\317\162\057\035\143\227\350\231\371
-\375\223\244\124\200\114\122\324\122\253\056\111\337\220\315\270
-\137\276\077\336\241\312\115\040\324\045\350\204\051\123\267\261
-\210\037\377\372\332\220\237\012\251\055\101\077\261\361\030\051
-\356\026\131\054\064\111\032\250\006\327\250\210\322\003\162\172
-\062\342\352\150\115\156\054\226\145\173\312\131\372\362\342\335
-\356\060\054\373\314\106\254\304\143\353\157\177\066\053\064\163
-\022\224\177\337\314\046\236\361\162\135\120\145\131\217\151\263
-\207\136\062\157\303\030\212\265\225\217\260\172\067\336\132\105
-\073\307\066\341\357\147\321\071\323\227\133\163\142\031\110\055
-\207\034\006\373\164\230\040\111\163\360\005\322\033\261\240\243
-\267\033\160\323\210\151\271\132\326\070\364\142\334\045\213\170
-\277\370\350\176\270\134\311\225\117\137\247\055\271\040\153\317
-\153\335\365\015\364\202\267\364\262\146\056\020\050\366\227\132
-\173\226\026\217\001\031\055\154\156\177\071\130\006\144\203\001
-\203\203\303\115\222\335\062\306\207\244\067\351\026\316\252\055
-\150\257\012\201\145\072\160\301\233\255\115\155\124\312\052\055
-\113\205\033\263\200\346\160\105\015\153\136\065\360\177\073\270
-\234\344\004\160\211\022\045\223\332\012\231\042\140\152\143\140
-\116\166\006\230\116\275\203\255\035\130\212\045\205\322\307\145
-\036\055\216\306\337\266\306\341\177\212\004\041\025\051\164\360
-\076\234\220\235\014\056\361\212\076\132\252\014\011\036\307\325
-\074\243\355\227\303\036\064\372\070\371\010\016\343\300\135\053
-\203\321\126\152\311\266\250\124\123\056\170\062\147\075\202\177
-\164\320\373\341\266\005\140\271\160\333\216\013\371\023\130\157
-\161\140\020\122\020\271\301\101\011\357\162\037\147\061\170\377
-\226\005\215\002\003\001\000\001\243\143\060\141\060\037\006\003
-\125\035\043\004\030\060\026\200\024\343\376\055\375\050\320\013
-\265\272\266\242\304\277\006\252\005\214\223\373\057\060\017\006
-\003\125\035\023\001\001\377\004\005\060\003\001\001\377\060\016
-\006\003\125\035\017\001\001\377\004\004\003\002\001\006\060\035
-\006\003\125\035\016\004\026\004\024\343\376\055\375\050\320\013
-\265\272\266\242\304\277\006\252\005\214\223\373\057\060\015\006
-\011\052\206\110\206\367\015\001\001\013\005\000\003\202\002\001
-\000\045\306\272\153\353\207\313\336\202\071\226\075\360\104\247
-\153\204\163\003\336\235\053\117\272\040\177\274\170\262\317\227
-\260\033\234\363\327\171\056\365\110\266\322\373\027\210\346\323
-\172\077\355\123\023\320\342\057\152\171\313\000\043\050\346\036
-\067\127\065\211\204\302\166\117\064\066\255\147\303\316\101\006
-\210\305\367\356\330\032\270\326\013\177\120\377\223\252\027\113
-\214\354\355\122\140\262\244\006\352\116\353\364\153\031\375\353
-\365\032\340\045\052\232\334\307\101\066\367\310\164\005\204\071
-\225\071\326\013\073\244\047\372\010\330\134\036\370\004\140\122
-\021\050\050\003\377\357\123\146\000\245\112\064\026\146\174\375
-\011\244\256\236\147\032\157\101\013\153\006\023\233\217\206\161
-\005\264\057\215\211\146\063\051\166\124\232\021\370\047\372\262
-\077\221\340\316\015\033\363\060\032\255\277\042\135\033\323\277
-\045\005\115\341\222\032\177\231\237\074\104\223\312\324\100\111
-\154\200\207\327\004\072\303\062\122\065\016\126\370\245\335\175
-\304\213\015\021\037\123\313\036\262\027\266\150\167\132\340\324
-\313\310\007\256\365\072\056\216\067\267\320\001\113\103\051\167
-\214\071\227\217\202\132\370\121\345\211\240\030\347\150\177\135
-\012\056\373\243\107\016\075\246\043\172\306\001\307\217\310\136
-\277\155\200\126\276\212\044\272\063\352\237\341\062\021\236\361
-\322\117\200\366\033\100\257\070\236\021\120\171\163\022\022\315
-\346\154\235\054\210\162\074\060\201\006\221\042\352\131\255\332
-\031\056\042\302\215\271\214\207\340\146\274\163\043\137\041\144
-\143\200\110\365\240\074\030\075\224\310\110\101\035\100\272\136
-\376\376\126\071\241\310\317\136\236\031\144\106\020\332\027\221
-\267\005\200\254\213\231\222\175\347\242\330\007\013\066\047\347
-\110\171\140\212\303\327\023\134\370\162\100\337\112\313\317\231
-\000\012\000\013\021\225\332\126\105\003\210\012\237\147\320\325
-\171\261\250\215\100\155\015\302\172\100\372\363\137\144\107\222
-\313\123\271\273\131\316\117\375\320\025\123\001\330\337\353\331
-\346\166\357\320\043\273\073\251\171\263\325\002\051\315\211\243
-\226\017\112\065\347\116\102\300\165\315\007\317\346\054\353\173
-\056
-END
-CKA_NSS_MOZILLA_CA_POLICY CK_BBOOL CK_TRUE
-CKA_NSS_SERVER_DISTRUST_AFTER CK_BBOOL CK_FALSE
-CKA_NSS_EMAIL_DISTRUST_AFTER CK_BBOOL CK_FALSE
-
-# Trust for "CFCA EV ROOT"
-# Issuer: CN=CFCA EV ROOT,O=China Financial Certification Authority,C=CN
-# Serial Number: 407555286 (0x184accd6)
-# Subject: CN=CFCA EV ROOT,O=China Financial Certification Authority,C=CN
-# Not Valid Before: Wed Aug 08 03:07:01 2012
-# Not Valid After : Mon Dec 31 03:07:01 2029
-# Fingerprint (SHA-256): 5C:C3:D7:8E:4E:1D:5E:45:54:7A:04:E6:87:3E:64:F9:0C:F9:53:6D:1C:CC:2E:F8:00:F3:55:C4:C5:FD:70:FD
-# Fingerprint (SHA1): E2:B8:29:4B:55:84:AB:6B:58:C2:90:46:6C:AC:3F:B8:39:8F:84:83
-CKA_CLASS CK_OBJECT_CLASS CKO_NSS_TRUST
-CKA_TOKEN CK_BBOOL CK_TRUE
-CKA_PRIVATE CK_BBOOL CK_FALSE
-CKA_MODIFIABLE CK_BBOOL CK_FALSE
-CKA_LABEL UTF8 "CFCA EV ROOT"
-CKA_CERT_SHA1_HASH MULTILINE_OCTAL
-\342\270\051\113\125\204\253\153\130\302\220\106\154\254\077\270
-\071\217\204\203
-END
-CKA_CERT_MD5_HASH MULTILINE_OCTAL
-\164\341\266\355\046\172\172\104\060\063\224\253\173\047\201\060
-END
-CKA_ISSUER MULTILINE_OCTAL
-\060\126\061\013\060\011\006\003\125\004\006\023\002\103\116\061
-\060\060\056\006\003\125\004\012\014\047\103\150\151\156\141\040
-\106\151\156\141\156\143\151\141\154\040\103\145\162\164\151\146
-\151\143\141\164\151\157\156\040\101\165\164\150\157\162\151\164
-\171\061\025\060\023\006\003\125\004\003\014\014\103\106\103\101
-\040\105\126\040\122\117\117\124
-END
-CKA_SERIAL_NUMBER MULTILINE_OCTAL
-\002\004\030\112\314\326
-END
-CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
-CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
-CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
-CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
-
 #
 # Certificate "OISTE WISeKey Global Root GB CA"
 #
@@ -12690,176 +12522,6 @@ CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
 CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
 CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
 
-#
-# Certificate "GDCA TrustAUTH R5 ROOT"
-#
-# Issuer: CN=GDCA TrustAUTH R5 ROOT,O="GUANG DONG CERTIFICATE AUTHORITY CO.,LTD.",C=CN
-# Serial Number:7d:09:97:fe:f0:47:ea:7a
-# Subject: CN=GDCA TrustAUTH R5 ROOT,O="GUANG DONG CERTIFICATE AUTHORITY CO.,LTD.",C=CN
-# Not Valid Before: Wed Nov 26 05:13:15 2014
-# Not Valid After : Mon Dec 31 15:59:59 2040
-# Fingerprint (SHA-256): BF:FF:8F:D0:44:33:48:7D:6A:8A:A6:0C:1A:29:76:7A:9F:C2:BB:B0:5E:42:0F:71:3A:13:B9:92:89:1D:38:93
-# Fingerprint (SHA1): 0F:36:38:5B:81:1A:25:C3:9B:31:4E:83:CA:E9:34:66:70:CC:74:B4
-CKA_CLASS CK_OBJECT_CLASS CKO_CERTIFICATE
-CKA_TOKEN CK_BBOOL CK_TRUE
-CKA_PRIVATE CK_BBOOL CK_FALSE
-CKA_MODIFIABLE CK_BBOOL CK_FALSE
-CKA_LABEL UTF8 "GDCA TrustAUTH R5 ROOT"
-CKA_CERTIFICATE_TYPE CK_CERTIFICATE_TYPE CKC_X_509
-CKA_SUBJECT MULTILINE_OCTAL
-\060\142\061\013\060\011\006\003\125\004\006\023\002\103\116\061
-\062\060\060\006\003\125\004\012\014\051\107\125\101\116\107\040
-\104\117\116\107\040\103\105\122\124\111\106\111\103\101\124\105
-\040\101\125\124\110\117\122\111\124\131\040\103\117\056\054\114
-\124\104\056\061\037\060\035\006\003\125\004\003\014\026\107\104
-\103\101\040\124\162\165\163\164\101\125\124\110\040\122\065\040
-\122\117\117\124
-END
-CKA_ID UTF8 "0"
-CKA_ISSUER MULTILINE_OCTAL
-\060\142\061\013\060\011\006\003\125\004\006\023\002\103\116\061
-\062\060\060\006\003\125\004\012\014\051\107\125\101\116\107\040
-\104\117\116\107\040\103\105\122\124\111\106\111\103\101\124\105
-\040\101\125\124\110\117\122\111\124\131\040\103\117\056\054\114
-\124\104\056\061\037\060\035\006\003\125\004\003\014\026\107\104
-\103\101\040\124\162\165\163\164\101\125\124\110\040\122\065\040
-\122\117\117\124
-END
-CKA_SERIAL_NUMBER MULTILINE_OCTAL
-\002\010\175\011\227\376\360\107\352\172
-END
-CKA_VALUE MULTILINE_OCTAL
-\060\202\005\210\060\202\003\160\240\003\002\001\002\002\010\175
-\011\227\376\360\107\352\172\060\015\006\011\052\206\110\206\367
-\015\001\001\013\005\000\060\142\061\013\060\011\006\003\125\004
-\006\023\002\103\116\061\062\060\060\006\003\125\004\012\014\051
-\107\125\101\116\107\040\104\117\116\107\040\103\105\122\124\111
-\106\111\103\101\124\105\040\101\125\124\110\117\122\111\124\131
-\040\103\117\056\054\114\124\104\056\061\037\060\035\006\003\125
-\004\003\014\026\107\104\103\101\040\124\162\165\163\164\101\125
-\124\110\040\122\065\040\122\117\117\124\060\036\027\015\061\064
-\061\061\062\066\060\065\061\063\061\065\132\027\015\064\060\061
-\062\063\061\061\065\065\071\065\071\132\060\142\061\013\060\011
-\006\003\125\004\006\023\002\103\116\061\062\060\060\006\003\125
-\004\012\014\051\107\125\101\116\107\040\104\117\116\107\040\103
-\105\122\124\111\106\111\103\101\124\105\040\101\125\124\110\117
-\122\111\124\131\040\103\117\056\054\114\124\104\056\061\037\060
-\035\006\003\125\004\003\014\026\107\104\103\101\040\124\162\165
-\163\164\101\125\124\110\040\122\065\040\122\117\117\124\060\202
-\002\042\060\015\006\011\052\206\110\206\367\015\001\001\001\005
-\000\003\202\002\017\000\060\202\002\012\002\202\002\001\000\331
-\243\026\360\310\164\164\167\233\357\063\015\073\006\176\125\374
-\265\140\217\166\206\022\102\175\126\146\076\210\202\355\162\143
-\016\236\213\335\064\054\002\121\121\303\031\375\131\124\204\311
-\361\153\263\114\260\351\350\106\135\070\306\242\247\056\021\127
-\272\202\025\242\234\217\155\260\231\112\012\362\353\211\160\143
-\116\171\304\267\133\275\242\135\261\362\101\002\053\255\251\072
-\243\354\171\012\354\137\072\343\375\357\200\074\255\064\233\032
-\253\210\046\173\126\242\202\206\037\353\065\211\203\177\137\256
-\051\116\075\266\156\354\256\301\360\047\233\256\343\364\354\357
-\256\177\367\206\075\162\172\353\245\373\131\116\247\353\225\214
-\042\071\171\341\055\010\217\314\274\221\270\101\367\024\301\043
-\251\303\255\232\105\104\263\262\327\054\315\306\051\342\120\020
-\256\134\313\202\216\027\030\066\175\227\346\210\232\260\115\064
-\011\364\054\271\132\146\052\260\027\233\236\036\166\235\112\146
-\061\101\337\077\373\305\006\357\033\266\176\032\106\066\367\144
-\143\073\343\071\030\043\347\147\165\024\325\165\127\222\067\275
-\276\152\033\046\120\362\066\046\006\220\305\160\001\144\155\166
-\146\341\221\333\156\007\300\141\200\056\262\056\057\214\160\247
-\321\073\074\263\221\344\156\266\304\073\160\362\154\222\227\011
-\315\107\175\030\300\363\273\236\017\326\213\256\007\266\132\017
-\316\013\014\107\247\345\076\270\275\175\307\233\065\240\141\227
-\072\101\165\027\314\053\226\167\052\222\041\036\331\225\166\040
-\147\150\317\015\275\337\326\037\011\152\232\342\314\163\161\244
-\057\175\022\200\267\123\060\106\136\113\124\231\017\147\311\245
-\310\362\040\301\202\354\235\021\337\302\002\373\032\073\321\355
-\040\232\357\145\144\222\020\015\052\342\336\160\361\030\147\202
-\214\141\336\270\274\321\057\234\373\017\320\053\355\033\166\271
-\344\071\125\370\370\241\035\270\252\200\000\114\202\347\262\177
-\011\270\274\060\240\057\015\365\122\236\216\367\222\263\012\000
-\035\000\124\227\006\340\261\007\331\307\017\134\145\175\074\155
-\131\127\344\355\245\215\351\100\123\237\025\113\240\161\366\032
-\041\343\332\160\006\041\130\024\207\205\167\171\252\202\171\002
-\003\001\000\001\243\102\060\100\060\035\006\003\125\035\016\004
-\026\004\024\342\311\100\237\115\316\350\232\241\174\317\016\077
-\145\305\051\210\152\031\121\060\017\006\003\125\035\023\001\001
-\377\004\005\060\003\001\001\377\060\016\006\003\125\035\017\001
-\001\377\004\004\003\002\001\206\060\015\006\011\052\206\110\206
-\367\015\001\001\013\005\000\003\202\002\001\000\321\111\127\340
-\247\314\150\130\272\001\017\053\031\315\215\260\141\105\254\021
-\355\143\120\151\370\037\177\276\026\217\375\235\353\013\252\062
-\107\166\322\147\044\355\275\174\063\062\227\052\307\005\206\146
-\015\027\175\024\025\033\324\353\375\037\232\366\136\227\151\267
-\032\045\244\012\263\221\077\137\066\254\213\354\127\250\076\347
-\201\212\030\127\071\205\164\032\102\307\351\133\023\137\217\371
-\010\351\222\164\215\365\107\322\253\073\326\373\170\146\116\066
-\175\371\351\222\351\004\336\375\111\143\374\155\373\024\161\223
-\147\057\107\112\267\271\377\036\052\163\160\106\060\277\132\362
-\057\171\245\341\215\014\331\371\262\143\067\214\067\145\205\160
-\152\134\133\011\162\271\255\143\074\261\335\370\374\062\277\067
-\206\344\273\216\230\047\176\272\037\026\341\160\021\362\003\337
-\045\142\062\047\046\030\062\204\237\377\000\072\023\272\232\115
-\364\117\270\024\160\042\261\312\053\220\316\051\301\160\364\057
-\235\177\362\220\036\326\132\337\267\106\374\346\206\372\313\340
-\040\166\172\272\246\313\365\174\336\142\245\261\213\356\336\202
-\146\212\116\072\060\037\077\200\313\255\047\272\014\136\327\320
-\261\126\312\167\161\262\265\165\241\120\251\100\103\027\302\050
-\331\317\122\213\133\310\143\324\102\076\240\063\172\106\056\367
-\012\040\106\124\176\152\117\061\361\201\176\102\164\070\145\163
-\047\356\306\174\270\216\327\245\072\327\230\241\234\214\020\125
-\323\333\113\354\100\220\362\315\156\127\322\142\016\174\127\223
-\261\247\155\315\235\203\273\052\347\345\266\073\161\130\255\375
-\321\105\274\132\221\356\123\025\157\323\105\011\165\156\272\220
-\135\036\004\317\067\337\036\250\146\261\214\346\040\152\357\374
-\110\116\164\230\102\257\051\157\056\152\307\373\175\321\146\061
-\042\314\206\000\176\146\203\014\102\364\275\064\222\303\032\352
-\117\312\176\162\115\013\160\214\246\110\273\246\241\024\366\373
-\130\104\231\024\256\252\013\223\151\240\051\045\112\245\313\053
-\335\212\146\007\026\170\025\127\161\033\354\365\107\204\363\236
-\061\067\172\325\177\044\255\344\274\375\375\314\156\203\350\014
-\250\267\101\154\007\335\275\074\206\227\057\322
-END
-CKA_NSS_MOZILLA_CA_POLICY CK_BBOOL CK_TRUE
-CKA_NSS_SERVER_DISTRUST_AFTER CK_BBOOL CK_FALSE
-CKA_NSS_EMAIL_DISTRUST_AFTER CK_BBOOL CK_FALSE
-
-# Trust for "GDCA TrustAUTH R5 ROOT"
-# Issuer: CN=GDCA TrustAUTH R5 ROOT,O="GUANG DONG CERTIFICATE AUTHORITY CO.,LTD.",C=CN
-# Serial Number:7d:09:97:fe:f0:47:ea:7a
-# Subject: CN=GDCA TrustAUTH R5 ROOT,O="GUANG DONG CERTIFICATE AUTHORITY CO.,LTD.",C=CN
-# Not Valid Before: Wed Nov 26 05:13:15 2014
-# Not Valid After : Mon Dec 31 15:59:59 2040
-# Fingerprint (SHA-256): BF:FF:8F:D0:44:33:48:7D:6A:8A:A6:0C:1A:29:76:7A:9F:C2:BB:B0:5E:42:0F:71:3A:13:B9:92:89:1D:38:93
-# Fingerprint (SHA1): 0F:36:38:5B:81:1A:25:C3:9B:31:4E:83:CA:E9:34:66:70:CC:74:B4
-CKA_CLASS CK_OBJECT_CLASS CKO_NSS_TRUST
-CKA_TOKEN CK_BBOOL CK_TRUE
-CKA_PRIVATE CK_BBOOL CK_FALSE
-CKA_MODIFIABLE CK_BBOOL CK_FALSE
-CKA_LABEL UTF8 "GDCA TrustAUTH R5 ROOT"
-CKA_CERT_SHA1_HASH MULTILINE_OCTAL
-\017\066\070\133\201\032\045\303\233\061\116\203\312\351\064\146
-\160\314\164\264
-END
-CKA_CERT_MD5_HASH MULTILINE_OCTAL
-\143\314\331\075\064\065\134\157\123\243\342\010\160\110\037\264
-END
-CKA_ISSUER MULTILINE_OCTAL
-\060\142\061\013\060\011\006\003\125\004\006\023\002\103\116\061
-\062\060\060\006\003\125\004\012\014\051\107\125\101\116\107\040
-\104\117\116\107\040\103\105\122\124\111\106\111\103\101\124\105
-\040\101\125\124\110\117\122\111\124\131\040\103\117\056\054\114
-\124\104\056\061\037\060\035\006\003\125\004\003\014\026\107\104
-\103\101\040\124\162\165\163\164\101\125\124\110\040\122\065\040
-\122\117\117\124
-END
-CKA_SERIAL_NUMBER MULTILINE_OCTAL
-\002\010\175\011\227\376\360\107\352\172
-END
-CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
-CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
-CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
-CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
-
 #
 # Certificate "SSL.com Root Certification Authority RSA"
 #
@@ -13762,328 +13424,6 @@ CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
 CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
 CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
 
-#
-# Certificate "UCA Global G2 Root"
-#
-# Issuer: CN=UCA Global G2 Root,O=UniTrust,C=CN
-# Serial Number:5d:df:b1:da:5a:a3:ed:5d:be:5a:65:20:65:03:90:ef
-# Subject: CN=UCA Global G2 Root,O=UniTrust,C=CN
-# Not Valid Before: Fri Mar 11 00:00:00 2016
-# Not Valid After : Mon Dec 31 00:00:00 2040
-# Fingerprint (SHA-256): 9B:EA:11:C9:76:FE:01:47:64:C1:BE:56:A6:F9:14:B5:A5:60:31:7A:BD:99:88:39:33:82:E5:16:1A:A0:49:3C
-# Fingerprint (SHA1): 28:F9:78:16:19:7A:FF:18:25:18:AA:44:FE:C1:A0:CE:5C:B6:4C:8A
-CKA_CLASS CK_OBJECT_CLASS CKO_CERTIFICATE
-CKA_TOKEN CK_BBOOL CK_TRUE
-CKA_PRIVATE CK_BBOOL CK_FALSE
-CKA_MODIFIABLE CK_BBOOL CK_FALSE
-CKA_LABEL UTF8 "UCA Global G2 Root"
-CKA_CERTIFICATE_TYPE CK_CERTIFICATE_TYPE CKC_X_509
-CKA_SUBJECT MULTILINE_OCTAL
-\060\075\061\013\060\011\006\003\125\004\006\023\002\103\116\061
-\021\060\017\006\003\125\004\012\014\010\125\156\151\124\162\165
-\163\164\061\033\060\031\006\003\125\004\003\014\022\125\103\101
-\040\107\154\157\142\141\154\040\107\062\040\122\157\157\164
-END
-CKA_ID UTF8 "0"
-CKA_ISSUER MULTILINE_OCTAL
-\060\075\061\013\060\011\006\003\125\004\006\023\002\103\116\061
-\021\060\017\006\003\125\004\012\014\010\125\156\151\124\162\165
-\163\164\061\033\060\031\006\003\125\004\003\014\022\125\103\101
-\040\107\154\157\142\141\154\040\107\062\040\122\157\157\164
-END
-CKA_SERIAL_NUMBER MULTILINE_OCTAL
-\002\020\135\337\261\332\132\243\355\135\276\132\145\040\145\003
-\220\357
-END
-CKA_VALUE MULTILINE_OCTAL
-\060\202\005\106\060\202\003\056\240\003\002\001\002\002\020\135
-\337\261\332\132\243\355\135\276\132\145\040\145\003\220\357\060
-\015\006\011\052\206\110\206\367\015\001\001\013\005\000\060\075
-\061\013\060\011\006\003\125\004\006\023\002\103\116\061\021\060
-\017\006\003\125\004\012\014\010\125\156\151\124\162\165\163\164
-\061\033\060\031\006\003\125\004\003\014\022\125\103\101\040\107
-\154\157\142\141\154\040\107\062\040\122\157\157\164\060\036\027
-\015\061\066\060\063\061\061\060\060\060\060\060\060\132\027\015
-\064\060\061\062\063\061\060\060\060\060\060\060\132\060\075\061
-\013\060\011\006\003\125\004\006\023\002\103\116\061\021\060\017
-\006\003\125\004\012\014\010\125\156\151\124\162\165\163\164\061
-\033\060\031\006\003\125\004\003\014\022\125\103\101\040\107\154
-\157\142\141\154\040\107\062\040\122\157\157\164\060\202\002\042
-\060\015\006\011\052\206\110\206\367\015\001\001\001\005\000\003
-\202\002\017\000\060\202\002\012\002\202\002\001\000\305\346\053
-\157\174\357\046\005\047\243\201\044\332\157\313\001\371\231\232
-\251\062\302\042\207\141\101\221\073\313\303\150\033\006\305\114
-\251\053\301\147\027\042\035\053\355\371\051\211\223\242\170\275
-\222\153\240\243\015\242\176\312\223\263\246\321\214\065\325\165
-\371\027\366\317\105\305\345\172\354\167\223\240\217\043\256\016
-\032\003\177\276\324\320\355\056\173\253\106\043\133\377\054\346
-\124\172\224\300\052\025\360\311\215\260\172\073\044\341\327\150
-\342\061\074\006\063\106\266\124\021\246\245\057\042\124\052\130
-\015\001\002\361\372\025\121\147\154\300\372\327\266\033\177\321
-\126\210\057\032\072\215\073\273\202\021\340\107\000\320\122\207
-\253\373\206\176\017\044\153\100\235\064\147\274\215\307\055\206
-\157\171\076\216\251\074\027\113\177\260\231\343\260\161\140\334
-\013\365\144\303\316\103\274\155\161\271\322\336\047\133\212\350
-\330\306\256\341\131\175\317\050\055\065\270\225\126\032\361\262
-\130\113\267\022\067\310\174\263\355\113\200\341\215\372\062\043
-\266\157\267\110\225\010\261\104\116\205\214\072\002\124\040\057
-\337\277\127\117\073\072\220\041\327\301\046\065\124\040\354\307
-\077\107\354\357\132\277\113\172\301\255\073\027\120\134\142\330
-\017\113\112\334\053\372\156\274\163\222\315\354\307\120\350\101
-\226\327\251\176\155\330\351\035\217\212\265\271\130\222\272\112
-\222\053\014\126\375\200\353\010\360\136\051\156\033\034\014\257
-\217\223\211\255\333\275\243\236\041\312\211\031\354\337\265\303
-\032\353\026\376\170\066\114\326\156\320\076\027\034\220\027\153
-\046\272\373\172\057\277\021\034\030\016\055\163\003\217\240\345
-\065\240\132\342\114\165\035\161\341\071\070\123\170\100\314\203
-\223\327\012\236\235\133\217\212\344\345\340\110\344\110\262\107
-\315\116\052\165\052\173\362\042\366\311\276\011\221\226\127\172
-\210\210\254\356\160\254\371\334\051\343\014\034\073\022\116\104
-\326\247\116\260\046\310\363\331\032\227\221\150\352\357\215\106
-\006\322\126\105\130\232\074\014\017\203\270\005\045\303\071\317
-\073\244\064\211\267\171\022\057\107\305\347\251\227\151\374\246
-\167\147\265\337\173\361\172\145\025\344\141\126\145\002\003\001
-\000\001\243\102\060\100\060\016\006\003\125\035\017\001\001\377
-\004\004\003\002\001\006\060\017\006\003\125\035\023\001\001\377
-\004\005\060\003\001\001\377\060\035\006\003\125\035\016\004\026
-\004\024\201\304\214\314\365\344\060\377\245\014\010\137\214\025
-\147\041\164\001\337\337\060\015\006\011\052\206\110\206\367\015
-\001\001\013\005\000\003\202\002\001\000\023\145\042\365\216\053
-\255\104\344\313\377\271\150\346\303\200\110\075\004\173\372\043
-\057\172\355\066\332\262\316\155\366\346\236\345\137\130\217\313
-\067\062\241\310\145\266\256\070\075\065\033\076\274\073\266\004
-\320\274\371\111\365\233\367\205\305\066\266\313\274\370\310\071
-\325\344\137\007\275\025\124\227\164\312\312\355\117\272\272\144
-\166\237\201\270\204\105\111\114\215\157\242\353\261\314\321\303
-\224\332\104\302\346\342\352\030\350\242\037\047\005\272\327\345
-\326\251\315\335\357\166\230\215\000\016\315\033\372\003\267\216
-\200\130\016\047\077\122\373\224\242\312\136\145\311\326\204\332
-\271\065\161\363\046\300\117\167\346\201\047\322\167\073\232\024
-\157\171\364\366\320\341\323\224\272\320\127\121\275\047\005\015
-\301\375\310\022\060\356\157\215\021\053\010\235\324\324\277\200
-\105\024\232\210\104\332\060\352\264\247\343\356\357\133\202\325
-\076\326\255\170\222\333\134\074\363\330\255\372\270\153\177\304
-\066\050\266\002\025\212\124\054\234\260\027\163\216\320\067\243
-\024\074\230\225\000\014\051\005\133\236\111\111\261\137\307\343
-\313\317\047\145\216\065\027\267\127\310\060\331\101\133\271\024
-\266\350\302\017\224\061\247\224\230\314\152\353\265\341\047\365
-\020\250\001\350\216\022\142\350\210\314\265\177\106\227\300\233
-\020\146\070\032\066\106\137\042\150\075\337\311\306\023\047\253
-\123\006\254\242\074\206\006\145\157\261\176\261\051\104\232\243
-\272\111\151\050\151\217\327\345\137\255\004\206\144\157\032\240
-\014\305\010\142\316\200\243\320\363\354\150\336\276\063\307\027
-\133\177\200\304\114\114\261\246\204\212\303\073\270\011\315\024
-\201\272\030\343\124\127\066\376\333\057\174\107\241\072\063\310
-\371\130\073\104\117\261\312\002\211\004\226\050\150\305\113\270
-\046\211\273\326\063\057\120\325\376\232\211\272\030\062\222\124
-\306\133\340\235\371\136\345\015\042\233\366\332\342\310\041\262
-\142\041\252\206\100\262\056\144\323\137\310\343\176\021\147\105
-\037\005\376\343\242\357\263\250\263\363\175\217\370\014\037\042
-\037\055\160\264\270\001\064\166\060\000\345\043\170\247\126\327
-\120\037\212\373\006\365\302\031\360\320
-END
-CKA_NSS_MOZILLA_CA_POLICY CK_BBOOL CK_TRUE
-CKA_NSS_SERVER_DISTRUST_AFTER CK_BBOOL CK_FALSE
-CKA_NSS_EMAIL_DISTRUST_AFTER CK_BBOOL CK_FALSE
-
-# Trust for "UCA Global G2 Root"
-# Issuer: CN=UCA Global G2 Root,O=UniTrust,C=CN
-# Serial Number:5d:df:b1:da:5a:a3:ed:5d:be:5a:65:20:65:03:90:ef
-# Subject: CN=UCA Global G2 Root,O=UniTrust,C=CN
-# Not Valid Before: Fri Mar 11 00:00:00 2016
-# Not Valid After : Mon Dec 31 00:00:00 2040
-# Fingerprint (SHA-256): 9B:EA:11:C9:76:FE:01:47:64:C1:BE:56:A6:F9:14:B5:A5:60:31:7A:BD:99:88:39:33:82:E5:16:1A:A0:49:3C
-# Fingerprint (SHA1): 28:F9:78:16:19:7A:FF:18:25:18:AA:44:FE:C1:A0:CE:5C:B6:4C:8A
-CKA_CLASS CK_OBJECT_CLASS CKO_NSS_TRUST
-CKA_TOKEN CK_BBOOL CK_TRUE
-CKA_PRIVATE CK_BBOOL CK_FALSE
-CKA_MODIFIABLE CK_BBOOL CK_FALSE
-CKA_LABEL UTF8 "UCA Global G2 Root"
-CKA_CERT_SHA1_HASH MULTILINE_OCTAL
-\050\371\170\026\031\172\377\030\045\030\252\104\376\301\240\316
-\134\266\114\212
-END
-CKA_CERT_MD5_HASH MULTILINE_OCTAL
-\200\376\360\304\112\360\134\142\062\237\034\272\170\251\120\370
-END
-CKA_ISSUER MULTILINE_OCTAL
-\060\075\061\013\060\011\006\003\125\004\006\023\002\103\116\061
-\021\060\017\006\003\125\004\012\014\010\125\156\151\124\162\165
-\163\164\061\033\060\031\006\003\125\004\003\014\022\125\103\101
-\040\107\154\157\142\141\154\040\107\062\040\122\157\157\164
-END
-CKA_SERIAL_NUMBER MULTILINE_OCTAL
-\002\020\135\337\261\332\132\243\355\135\276\132\145\040\145\003
-\220\357
-END
-CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
-CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
-CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
-CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
-
-#
-# Certificate "UCA Extended Validation Root"
-#
-# Issuer: CN=UCA Extended Validation Root,O=UniTrust,C=CN
-# Serial Number:4f:d2:2b:8f:f5:64:c8:33:9e:4f:34:58:66:23:70:60
-# Subject: CN=UCA Extended Validation Root,O=UniTrust,C=CN
-# Not Valid Before: Fri Mar 13 00:00:00 2015
-# Not Valid After : Fri Dec 31 00:00:00 2038
-# Fingerprint (SHA-256): D4:3A:F9:B3:54:73:75:5C:96:84:FC:06:D7:D8:CB:70:EE:5C:28:E7:73:FB:29:4E:B4:1E:E7:17:22:92:4D:24
-# Fingerprint (SHA1): A3:A1:B0:6F:24:61:23:4A:E3:36:A5:C2:37:FC:A6:FF:DD:F0:D7:3A
-CKA_CLASS CK_OBJECT_CLASS CKO_CERTIFICATE
-CKA_TOKEN CK_BBOOL CK_TRUE
-CKA_PRIVATE CK_BBOOL CK_FALSE
-CKA_MODIFIABLE CK_BBOOL CK_FALSE
-CKA_LABEL UTF8 "UCA Extended Validation Root"
-CKA_CERTIFICATE_TYPE CK_CERTIFICATE_TYPE CKC_X_509
-CKA_SUBJECT MULTILINE_OCTAL
-\060\107\061\013\060\011\006\003\125\004\006\023\002\103\116\061
-\021\060\017\006\003\125\004\012\014\010\125\156\151\124\162\165
-\163\164\061\045\060\043\006\003\125\004\003\014\034\125\103\101
-\040\105\170\164\145\156\144\145\144\040\126\141\154\151\144\141
-\164\151\157\156\040\122\157\157\164
-END
-CKA_ID UTF8 "0"
-CKA_ISSUER MULTILINE_OCTAL
-\060\107\061\013\060\011\006\003\125\004\006\023\002\103\116\061
-\021\060\017\006\003\125\004\012\014\010\125\156\151\124\162\165
-\163\164\061\045\060\043\006\003\125\004\003\014\034\125\103\101
-\040\105\170\164\145\156\144\145\144\040\126\141\154\151\144\141
-\164\151\157\156\040\122\157\157\164
-END
-CKA_SERIAL_NUMBER MULTILINE_OCTAL
-\002\020\117\322\053\217\365\144\310\063\236\117\064\130\146\043
-\160\140
-END
-CKA_VALUE MULTILINE_OCTAL
-\060\202\005\132\060\202\003\102\240\003\002\001\002\002\020\117
-\322\053\217\365\144\310\063\236\117\064\130\146\043\160\140\060
-\015\006\011\052\206\110\206\367\015\001\001\013\005\000\060\107
-\061\013\060\011\006\003\125\004\006\023\002\103\116\061\021\060
-\017\006\003\125\004\012\014\010\125\156\151\124\162\165\163\164
-\061\045\060\043\006\003\125\004\003\014\034\125\103\101\040\105
-\170\164\145\156\144\145\144\040\126\141\154\151\144\141\164\151
-\157\156\040\122\157\157\164\060\036\027\015\061\065\060\063\061
-\063\060\060\060\060\060\060\132\027\015\063\070\061\062\063\061
-\060\060\060\060\060\060\132\060\107\061\013\060\011\006\003\125
-\004\006\023\002\103\116\061\021\060\017\006\003\125\004\012\014
-\010\125\156\151\124\162\165\163\164\061\045\060\043\006\003\125
-\004\003\014\034\125\103\101\040\105\170\164\145\156\144\145\144
-\040\126\141\154\151\144\141\164\151\157\156\040\122\157\157\164
-\060\202\002\042\060\015\006\011\052\206\110\206\367\015\001\001
-\001\005\000\003\202\002\017\000\060\202\002\012\002\202\002\001
-\000\251\011\007\050\023\002\260\231\340\144\252\036\103\026\172
-\163\261\221\240\165\076\250\372\343\070\000\172\354\211\152\040
-\017\213\305\260\233\063\003\132\206\306\130\206\325\301\205\273
-\117\306\234\100\115\312\276\356\151\226\270\255\201\060\232\174
-\222\005\353\005\053\232\110\320\270\166\076\226\310\040\273\322
-\260\361\217\330\254\105\106\377\252\147\140\264\167\176\152\037
-\074\032\122\172\004\075\007\074\205\015\204\320\037\166\012\367
-\152\024\337\162\343\064\174\127\116\126\001\076\171\361\252\051
-\073\154\372\370\217\155\115\310\065\337\256\353\334\044\356\171
-\105\247\205\266\005\210\336\210\135\045\174\227\144\147\011\331
-\277\132\025\005\206\363\011\036\354\130\062\063\021\363\167\144
-\260\166\037\344\020\065\027\033\362\016\261\154\244\052\243\163
-\374\011\037\036\062\031\123\021\347\331\263\054\056\166\056\241
-\243\336\176\152\210\011\350\362\007\212\370\262\315\020\347\342
-\163\100\223\273\010\321\077\341\374\013\224\263\045\357\174\246
-\327\321\257\237\377\226\232\365\221\173\230\013\167\324\176\350
-\007\322\142\265\225\071\343\363\361\155\017\016\145\204\212\143
-\124\305\200\266\340\236\113\175\107\046\247\001\010\135\321\210
-\236\327\303\062\104\372\202\112\012\150\124\177\070\123\003\314
-\244\000\063\144\121\131\013\243\202\221\172\136\354\026\302\363
-\052\346\142\332\052\333\131\142\020\045\112\052\201\013\107\007
-\103\006\160\207\322\372\223\021\051\172\110\115\353\224\307\160
-\115\257\147\325\121\261\200\040\001\001\264\172\010\246\220\177
-\116\340\357\007\101\207\257\152\245\136\213\373\317\120\262\232
-\124\257\303\211\272\130\055\365\060\230\261\066\162\071\176\111
-\004\375\051\247\114\171\344\005\127\333\224\271\026\123\215\106
-\263\035\225\141\127\126\177\257\360\026\133\141\130\157\066\120
-\021\013\330\254\053\225\026\032\016\037\010\315\066\064\145\020
-\142\146\325\200\137\024\040\137\055\014\240\170\012\150\326\054
-\327\351\157\053\322\112\005\223\374\236\157\153\147\377\210\361
-\116\245\151\112\122\067\005\352\306\026\215\322\304\231\321\202
-\053\073\272\065\165\367\121\121\130\363\310\007\335\344\264\003
-\177\002\003\001\000\001\243\102\060\100\060\035\006\003\125\035
-\016\004\026\004\024\331\164\072\344\060\075\015\367\022\334\176
-\132\005\237\036\064\232\367\341\024\060\017\006\003\125\035\023
-\001\001\377\004\005\060\003\001\001\377\060\016\006\003\125\035
-\017\001\001\377\004\004\003\002\001\206\060\015\006\011\052\206
-\110\206\367\015\001\001\013\005\000\003\202\002\001\000\066\215
-\227\314\102\025\144\051\067\233\046\054\326\373\256\025\151\054
-\153\032\032\367\137\266\371\007\114\131\352\363\311\310\271\256
-\314\272\056\172\334\300\365\260\055\300\073\257\237\160\005\021
-\152\237\045\117\001\051\160\343\345\014\341\352\132\174\334\111
-\273\301\036\052\201\365\026\113\162\221\310\242\061\271\252\332
-\374\235\037\363\135\100\002\023\374\116\034\006\312\263\024\220
-\124\027\031\022\032\361\037\327\014\151\132\366\161\170\364\224
-\175\221\013\216\354\220\124\216\274\157\241\114\253\374\164\144
-\375\161\232\370\101\007\241\315\221\344\074\232\340\233\062\071
-\163\253\052\325\151\310\170\221\046\061\175\342\307\060\361\374
-\024\170\167\022\016\023\364\335\026\224\277\113\147\173\160\123
-\205\312\260\273\363\070\115\054\220\071\300\015\302\135\153\351
-\342\345\325\210\215\326\054\277\253\033\276\265\050\207\022\027
-\164\156\374\175\374\217\320\207\046\260\033\373\271\154\253\342
-\236\075\025\301\073\056\147\002\130\221\237\357\370\102\037\054
-\267\150\365\165\255\317\265\366\377\021\175\302\360\044\245\255
-\323\372\240\074\251\372\135\334\245\240\357\104\244\276\326\350
-\345\344\023\226\027\173\006\076\062\355\307\267\102\274\166\243
-\330\145\070\053\070\065\121\041\016\016\157\056\064\023\100\341
-\053\147\014\155\112\101\060\030\043\132\062\125\231\311\027\340
-\074\336\366\354\171\255\053\130\031\242\255\054\042\032\225\216
-\276\226\220\135\102\127\304\371\024\003\065\053\034\055\121\127
-\010\247\072\336\077\344\310\264\003\163\302\301\046\200\273\013
-\102\037\255\015\257\046\162\332\314\276\263\243\203\130\015\202
-\305\037\106\121\343\234\030\314\215\233\215\354\111\353\165\120
-\325\214\050\131\312\164\064\332\214\013\041\253\036\352\033\345
-\307\375\025\076\300\027\252\373\043\156\046\106\313\372\371\261
-\162\153\151\317\042\204\013\142\017\254\331\031\000\224\242\166
-\074\324\055\232\355\004\236\055\006\142\020\067\122\034\205\162
-\033\047\345\314\306\061\354\067\354\143\131\233\013\035\166\314
-\176\062\232\210\225\010\066\122\273\336\166\137\166\111\111\255
-\177\275\145\040\262\311\301\053\166\030\166\237\126\261
-END
-CKA_NSS_MOZILLA_CA_POLICY CK_BBOOL CK_TRUE
-CKA_NSS_SERVER_DISTRUST_AFTER CK_BBOOL CK_FALSE
-CKA_NSS_EMAIL_DISTRUST_AFTER CK_BBOOL CK_FALSE
-
-# Trust for "UCA Extended Validation Root"
-# Issuer: CN=UCA Extended Validation Root,O=UniTrust,C=CN
-# Serial Number:4f:d2:2b:8f:f5:64:c8:33:9e:4f:34:58:66:23:70:60
-# Subject: CN=UCA Extended Validation Root,O=UniTrust,C=CN
-# Not Valid Before: Fri Mar 13 00:00:00 2015
-# Not Valid After : Fri Dec 31 00:00:00 2038
-# Fingerprint (SHA-256): D4:3A:F9:B3:54:73:75:5C:96:84:FC:06:D7:D8:CB:70:EE:5C:28:E7:73:FB:29:4E:B4:1E:E7:17:22:92:4D:24
-# Fingerprint (SHA1): A3:A1:B0:6F:24:61:23:4A:E3:36:A5:C2:37:FC:A6:FF:DD:F0:D7:3A
-CKA_CLASS CK_OBJECT_CLASS CKO_NSS_TRUST
-CKA_TOKEN CK_BBOOL CK_TRUE
-CKA_PRIVATE CK_BBOOL CK_FALSE
-CKA_MODIFIABLE CK_BBOOL CK_FALSE
-CKA_LABEL UTF8 "UCA Extended Validation Root"
-CKA_CERT_SHA1_HASH MULTILINE_OCTAL
-\243\241\260\157\044\141\043\112\343\066\245\302\067\374\246\377
-\335\360\327\072
-END
-CKA_CERT_MD5_HASH MULTILINE_OCTAL
-\241\363\137\103\306\064\233\332\277\214\176\005\123\255\226\342
-END
-CKA_ISSUER MULTILINE_OCTAL
-\060\107\061\013\060\011\006\003\125\004\006\023\002\103\116\061
-\021\060\017\006\003\125\004\012\014\010\125\156\151\124\162\165
-\163\164\061\045\060\043\006\003\125\004\003\014\034\125\103\101
-\040\105\170\164\145\156\144\145\144\040\126\141\154\151\144\141
-\164\151\157\156\040\122\157\157\164
-END
-CKA_SERIAL_NUMBER MULTILINE_OCTAL
-\002\020\117\322\053\217\365\144\310\063\236\117\064\130\146\043
-\160\140
-END
-CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
-CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
-CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
-CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
-
 #
 # Certificate "Certigna Root CA"
 #
@@ -18624,280 +17964,6 @@ CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
 CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
 CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
 
-#
-# Certificate "vTrus ECC Root CA"
-#
-# Issuer: CN=vTrus ECC Root CA,O="iTrusChina Co.,Ltd.",C=CN
-# Serial Number:6e:6a:bc:59:aa:53:be:98:39:67:a2:d2:6b:a4:3b:e6:6d:1c:d6:da
-# Subject: CN=vTrus ECC Root CA,O="iTrusChina Co.,Ltd.",C=CN
-# Not Valid Before: Tue Jul 31 07:26:44 2018
-# Not Valid After : Fri Jul 31 07:26:44 2043
-# Fingerprint (SHA-256): 30:FB:BA:2C:32:23:8E:2A:98:54:7A:F9:79:31:E5:50:42:8B:9B:3F:1C:8E:EB:66:33:DC:FA:86:C5:B2:7D:D3
-# Fingerprint (SHA1): F6:9C:DB:B0:FC:F6:02:13:B6:52:32:A6:A3:91:3F:16:70:DA:C3:E1
-CKA_CLASS CK_OBJECT_CLASS CKO_CERTIFICATE
-CKA_TOKEN CK_BBOOL CK_TRUE
-CKA_PRIVATE CK_BBOOL CK_FALSE
-CKA_MODIFIABLE CK_BBOOL CK_FALSE
-CKA_LABEL UTF8 "vTrus ECC Root CA"
-CKA_CERTIFICATE_TYPE CK_CERTIFICATE_TYPE CKC_X_509
-CKA_SUBJECT MULTILINE_OCTAL
-\060\107\061\013\060\011\006\003\125\004\006\023\002\103\116\061
-\034\060\032\006\003\125\004\012\023\023\151\124\162\165\163\103
-\150\151\156\141\040\103\157\056\054\114\164\144\056\061\032\060
-\030\006\003\125\004\003\023\021\166\124\162\165\163\040\105\103
-\103\040\122\157\157\164\040\103\101
-END
-CKA_ID UTF8 "0"
-CKA_ISSUER MULTILINE_OCTAL
-\060\107\061\013\060\011\006\003\125\004\006\023\002\103\116\061
-\034\060\032\006\003\125\004\012\023\023\151\124\162\165\163\103
-\150\151\156\141\040\103\157\056\054\114\164\144\056\061\032\060
-\030\006\003\125\004\003\023\021\166\124\162\165\163\040\105\103
-\103\040\122\157\157\164\040\103\101
-END
-CKA_SERIAL_NUMBER MULTILINE_OCTAL
-\002\024\156\152\274\131\252\123\276\230\071\147\242\322\153\244
-\073\346\155\034\326\332
-END
-CKA_VALUE MULTILINE_OCTAL
-\060\202\002\017\060\202\001\225\240\003\002\001\002\002\024\156
-\152\274\131\252\123\276\230\071\147\242\322\153\244\073\346\155
-\034\326\332\060\012\006\010\052\206\110\316\075\004\003\003\060
-\107\061\013\060\011\006\003\125\004\006\023\002\103\116\061\034
-\060\032\006\003\125\004\012\023\023\151\124\162\165\163\103\150
-\151\156\141\040\103\157\056\054\114\164\144\056\061\032\060\030
-\006\003\125\004\003\023\021\166\124\162\165\163\040\105\103\103
-\040\122\157\157\164\040\103\101\060\036\027\015\061\070\060\067
-\063\061\060\067\062\066\064\064\132\027\015\064\063\060\067\063
-\061\060\067\062\066\064\064\132\060\107\061\013\060\011\006\003
-\125\004\006\023\002\103\116\061\034\060\032\006\003\125\004\012
-\023\023\151\124\162\165\163\103\150\151\156\141\040\103\157\056
-\054\114\164\144\056\061\032\060\030\006\003\125\004\003\023\021
-\166\124\162\165\163\040\105\103\103\040\122\157\157\164\040\103
-\101\060\166\060\020\006\007\052\206\110\316\075\002\001\006\005
-\053\201\004\000\042\003\142\000\004\145\120\112\256\214\171\226
-\112\252\034\010\303\243\242\315\376\131\126\101\167\375\046\224
-\102\273\035\315\010\333\163\262\133\165\363\317\234\116\202\364
-\277\370\141\046\205\154\326\205\133\162\160\322\375\333\142\264
-\337\123\213\275\261\104\130\142\102\011\307\372\177\133\020\347
-\376\100\375\300\330\303\053\062\347\160\246\267\246\040\125\035
-\173\200\135\113\217\147\114\361\020\243\102\060\100\060\035\006
-\003\125\035\016\004\026\004\024\230\071\315\276\330\262\214\367
-\262\253\341\255\044\257\173\174\241\333\037\317\060\017\006\003
-\125\035\023\001\001\377\004\005\060\003\001\001\377\060\016\006
-\003\125\035\017\001\001\377\004\004\003\002\001\006\060\012\006
-\010\052\206\110\316\075\004\003\003\003\150\000\060\145\002\060
-\127\235\335\126\361\307\343\351\270\111\120\153\233\151\303\157
-\354\303\175\045\344\127\225\023\100\233\122\323\073\363\100\031
-\274\046\307\055\006\236\265\173\066\237\365\045\324\143\153\000
-\002\061\000\351\323\306\236\126\232\052\314\241\332\077\310\146
-\053\323\130\234\040\205\372\253\221\212\160\160\021\070\140\144
-\013\142\011\221\130\000\371\115\373\064\150\332\011\255\041\006
-\030\224\316
-END
-CKA_NSS_MOZILLA_CA_POLICY CK_BBOOL CK_TRUE
-CKA_NSS_SERVER_DISTRUST_AFTER CK_BBOOL CK_FALSE
-CKA_NSS_EMAIL_DISTRUST_AFTER CK_BBOOL CK_FALSE
-
-# Trust for "vTrus ECC Root CA"
-# Issuer: CN=vTrus ECC Root CA,O="iTrusChina Co.,Ltd.",C=CN
-# Serial Number:6e:6a:bc:59:aa:53:be:98:39:67:a2:d2:6b:a4:3b:e6:6d:1c:d6:da
-# Subject: CN=vTrus ECC Root CA,O="iTrusChina Co.,Ltd.",C=CN
-# Not Valid Before: Tue Jul 31 07:26:44 2018
-# Not Valid After : Fri Jul 31 07:26:44 2043
-# Fingerprint (SHA-256): 30:FB:BA:2C:32:23:8E:2A:98:54:7A:F9:79:31:E5:50:42:8B:9B:3F:1C:8E:EB:66:33:DC:FA:86:C5:B2:7D:D3
-# Fingerprint (SHA1): F6:9C:DB:B0:FC:F6:02:13:B6:52:32:A6:A3:91:3F:16:70:DA:C3:E1
-CKA_CLASS CK_OBJECT_CLASS CKO_NSS_TRUST
-CKA_TOKEN CK_BBOOL CK_TRUE
-CKA_PRIVATE CK_BBOOL CK_FALSE
-CKA_MODIFIABLE CK_BBOOL CK_FALSE
-CKA_LABEL UTF8 "vTrus ECC Root CA"
-CKA_CERT_SHA1_HASH MULTILINE_OCTAL
-\366\234\333\260\374\366\002\023\266\122\062\246\243\221\077\026
-\160\332\303\341
-END
-CKA_CERT_MD5_HASH MULTILINE_OCTAL
-\336\113\301\365\122\214\233\103\341\076\217\125\124\027\215\205
-END
-CKA_ISSUER MULTILINE_OCTAL
-\060\107\061\013\060\011\006\003\125\004\006\023\002\103\116\061
-\034\060\032\006\003\125\004\012\023\023\151\124\162\165\163\103
-\150\151\156\141\040\103\157\056\054\114\164\144\056\061\032\060
-\030\006\003\125\004\003\023\021\166\124\162\165\163\040\105\103
-\103\040\122\157\157\164\040\103\101
-END
-CKA_SERIAL_NUMBER MULTILINE_OCTAL
-\002\024\156\152\274\131\252\123\276\230\071\147\242\322\153\244
-\073\346\155\034\326\332
-END
-CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
-CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
-CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
-CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
-
-#
-# Certificate "vTrus Root CA"
-#
-# Issuer: CN=vTrus Root CA,O="iTrusChina Co.,Ltd.",C=CN
-# Serial Number:43:e3:71:13:d8:b3:59:14:5d:b7:ce:8c:fd:35:fd:6f:bc:05:8d:45
-# Subject: CN=vTrus Root CA,O="iTrusChina Co.,Ltd.",C=CN
-# Not Valid Before: Tue Jul 31 07:24:05 2018
-# Not Valid After : Fri Jul 31 07:24:05 2043
-# Fingerprint (SHA-256): 8A:71:DE:65:59:33:6F:42:6C:26:E5:38:80:D0:0D:88:A1:8D:A4:C6:A9:1F:0D:CB:61:94:E2:06:C5:C9:63:87
-# Fingerprint (SHA1): 84:1A:69:FB:F5:CD:1A:25:34:13:3D:E3:F8:FC:B8:99:D0:C9:14:B7
-CKA_CLASS CK_OBJECT_CLASS CKO_CERTIFICATE
-CKA_TOKEN CK_BBOOL CK_TRUE
-CKA_PRIVATE CK_BBOOL CK_FALSE
-CKA_MODIFIABLE CK_BBOOL CK_FALSE
-CKA_LABEL UTF8 "vTrus Root CA"
-CKA_CERTIFICATE_TYPE CK_CERTIFICATE_TYPE CKC_X_509
-CKA_SUBJECT MULTILINE_OCTAL
-\060\103\061\013\060\011\006\003\125\004\006\023\002\103\116\061
-\034\060\032\006\003\125\004\012\023\023\151\124\162\165\163\103
-\150\151\156\141\040\103\157\056\054\114\164\144\056\061\026\060
-\024\006\003\125\004\003\023\015\166\124\162\165\163\040\122\157
-\157\164\040\103\101
-END
-CKA_ID UTF8 "0"
-CKA_ISSUER MULTILINE_OCTAL
-\060\103\061\013\060\011\006\003\125\004\006\023\002\103\116\061
-\034\060\032\006\003\125\004\012\023\023\151\124\162\165\163\103
-\150\151\156\141\040\103\157\056\054\114\164\144\056\061\026\060
-\024\006\003\125\004\003\023\015\166\124\162\165\163\040\122\157
-\157\164\040\103\101
-END
-CKA_SERIAL_NUMBER MULTILINE_OCTAL
-\002\024\103\343\161\023\330\263\131\024\135\267\316\214\375\065
-\375\157\274\005\215\105
-END
-CKA_VALUE MULTILINE_OCTAL
-\060\202\005\126\060\202\003\076\240\003\002\001\002\002\024\103
-\343\161\023\330\263\131\024\135\267\316\214\375\065\375\157\274
-\005\215\105\060\015\006\011\052\206\110\206\367\015\001\001\013
-\005\000\060\103\061\013\060\011\006\003\125\004\006\023\002\103
-\116\061\034\060\032\006\003\125\004\012\023\023\151\124\162\165
-\163\103\150\151\156\141\040\103\157\056\054\114\164\144\056\061
-\026\060\024\006\003\125\004\003\023\015\166\124\162\165\163\040
-\122\157\157\164\040\103\101\060\036\027\015\061\070\060\067\063
-\061\060\067\062\064\060\065\132\027\015\064\063\060\067\063\061
-\060\067\062\064\060\065\132\060\103\061\013\060\011\006\003\125
-\004\006\023\002\103\116\061\034\060\032\006\003\125\004\012\023
-\023\151\124\162\165\163\103\150\151\156\141\040\103\157\056\054
-\114\164\144\056\061\026\060\024\006\003\125\004\003\023\015\166
-\124\162\165\163\040\122\157\157\164\040\103\101\060\202\002\042
-\060\015\006\011\052\206\110\206\367\015\001\001\001\005\000\003
-\202\002\017\000\060\202\002\012\002\202\002\001\000\275\125\174
-\141\323\270\035\004\142\005\240\256\154\267\160\264\101\352\113
-\003\136\020\077\220\132\034\213\073\260\146\213\154\110\246\034
-\042\272\325\100\222\356\063\262\043\131\311\216\274\130\332\213
-\236\320\031\362\057\131\306\214\143\132\272\237\243\013\260\263
-\232\134\272\021\270\022\351\014\273\317\156\154\200\207\051\024
-\003\054\215\044\232\310\144\203\265\152\254\023\054\063\361\237
-\334\054\141\074\032\077\160\125\233\255\000\122\177\317\004\271
-\376\066\372\234\300\026\256\142\376\226\114\103\176\125\024\276
-\032\263\322\155\302\257\166\146\225\153\052\260\224\167\205\136
-\004\017\142\035\143\165\367\153\347\313\133\232\160\354\076\147
-\005\360\376\007\010\200\317\050\333\005\306\024\047\057\206\175
-\360\047\336\377\346\176\063\110\347\013\036\130\321\047\053\123
-\016\127\112\145\327\373\242\200\140\374\114\274\065\123\001\152
-\227\162\202\257\361\035\160\350\234\365\357\136\302\154\307\107
-\176\132\224\205\046\115\073\272\353\114\350\260\011\302\145\302
-\235\235\011\233\116\265\227\005\254\365\006\240\367\066\005\176
-\364\220\262\153\304\264\371\144\352\351\032\012\310\015\250\355
-\047\311\324\347\263\271\253\202\042\220\047\075\052\350\174\220
-\357\274\117\375\342\012\044\247\336\145\044\244\135\352\300\166
-\060\323\167\120\370\015\004\233\224\066\001\163\312\006\130\246
-\323\073\334\372\004\106\023\125\212\311\104\107\270\121\071\032
-\056\350\064\342\171\313\131\112\012\177\274\246\357\037\003\147
-\152\131\053\045\142\223\331\123\031\146\074\047\142\051\206\115
-\244\153\356\377\324\116\272\325\264\342\216\110\132\000\031\011
-\361\005\331\316\221\261\367\353\351\071\117\366\157\004\103\232
-\125\365\076\005\024\275\277\263\131\264\330\216\063\204\243\220
-\122\252\263\002\225\140\371\014\114\150\371\356\325\027\015\370
-\161\127\265\045\344\051\356\145\135\257\321\356\074\027\013\132
-\103\305\245\206\352\044\236\342\005\007\334\064\102\022\221\326
-\071\164\256\114\101\202\333\362\246\110\321\263\233\363\063\252
-\363\246\300\305\116\365\364\235\166\143\346\002\306\042\113\301
-\225\077\120\144\054\124\345\266\360\074\051\317\127\002\003\001
-\000\001\243\102\060\100\060\035\006\003\125\035\016\004\026\004
-\024\124\142\160\143\361\165\204\103\130\216\321\026\040\261\306
-\254\032\274\366\211\060\017\006\003\125\035\023\001\001\377\004
-\005\060\003\001\001\377\060\016\006\003\125\035\017\001\001\377
-\004\004\003\002\001\006\060\015\006\011\052\206\110\206\367\015
-\001\001\013\005\000\003\202\002\001\000\051\272\222\111\247\255
-\360\361\160\303\344\227\360\237\251\045\325\153\236\064\376\346
-\032\144\366\072\153\122\262\020\170\032\237\114\332\212\332\354
-\034\067\122\340\102\113\373\154\166\312\044\013\071\022\025\235
-\237\021\055\374\171\144\334\340\340\365\335\340\127\311\245\262
-\166\160\120\244\376\267\012\160\325\240\064\361\165\327\115\111
-\272\021\321\263\330\354\202\377\353\016\304\277\144\055\175\143
-\156\027\170\354\135\174\210\310\353\216\127\166\331\131\004\372
-\274\122\037\105\254\360\172\200\354\354\157\166\256\221\333\020
-\216\004\334\222\337\240\366\346\256\111\323\301\154\022\033\314
-\051\252\371\010\245\342\067\024\312\261\270\146\357\032\202\344
-\360\370\361\247\026\151\267\333\251\141\074\237\365\061\313\344
-\000\106\302\057\164\261\261\327\201\356\250\046\225\274\210\257
-\114\065\007\052\002\312\170\024\155\107\053\100\126\351\313\052
-\140\241\147\003\240\316\214\274\260\162\147\304\061\316\333\064
-\345\045\003\140\045\173\161\230\344\300\033\053\137\164\102\322
-\113\305\131\010\007\207\276\305\303\177\347\226\331\341\334\050
-\227\326\217\005\343\365\233\116\312\035\120\107\005\123\260\312
-\071\347\205\240\211\301\005\073\001\067\323\077\111\342\167\353
-\043\310\210\146\073\075\071\166\041\106\361\354\137\043\270\353
-\242\146\165\164\301\100\367\330\150\232\223\342\055\251\056\275
-\034\243\036\310\164\306\244\055\172\040\253\073\270\260\106\375
-\157\335\137\122\125\165\142\360\227\240\174\327\070\375\045\337
-\315\240\233\020\317\213\270\070\136\136\305\264\246\002\066\241
-\036\137\034\317\342\226\235\051\252\375\230\256\122\341\363\101
-\122\373\251\056\162\226\237\047\343\252\163\175\370\032\043\146
-\173\073\253\145\260\062\001\113\025\076\075\242\117\014\053\065
-\242\306\331\147\022\065\060\315\166\056\026\263\231\236\115\117
-\116\055\073\064\103\341\232\016\015\244\146\227\272\322\034\112
-\114\054\052\213\213\201\117\161\032\251\335\134\173\173\010\305
-\000\015\067\100\343\174\173\124\137\057\205\137\166\366\367\247
-\260\034\127\126\301\162\350\255\242\257\215\063\111\272\037\212
-\334\346\164\174\140\206\157\207\227\173
-END
-CKA_NSS_MOZILLA_CA_POLICY CK_BBOOL CK_TRUE
-CKA_NSS_SERVER_DISTRUST_AFTER CK_BBOOL CK_FALSE
-CKA_NSS_EMAIL_DISTRUST_AFTER CK_BBOOL CK_FALSE
-
-# Trust for "vTrus Root CA"
-# Issuer: CN=vTrus Root CA,O="iTrusChina Co.,Ltd.",C=CN
-# Serial Number:43:e3:71:13:d8:b3:59:14:5d:b7:ce:8c:fd:35:fd:6f:bc:05:8d:45
-# Subject: CN=vTrus Root CA,O="iTrusChina Co.,Ltd.",C=CN
-# Not Valid Before: Tue Jul 31 07:24:05 2018
-# Not Valid After : Fri Jul 31 07:24:05 2043
-# Fingerprint (SHA-256): 8A:71:DE:65:59:33:6F:42:6C:26:E5:38:80:D0:0D:88:A1:8D:A4:C6:A9:1F:0D:CB:61:94:E2:06:C5:C9:63:87
-# Fingerprint (SHA1): 84:1A:69:FB:F5:CD:1A:25:34:13:3D:E3:F8:FC:B8:99:D0:C9:14:B7
-CKA_CLASS CK_OBJECT_CLASS CKO_NSS_TRUST
-CKA_TOKEN CK_BBOOL CK_TRUE
-CKA_PRIVATE CK_BBOOL CK_FALSE
-CKA_MODIFIABLE CK_BBOOL CK_FALSE
-CKA_LABEL UTF8 "vTrus Root CA"
-CKA_CERT_SHA1_HASH MULTILINE_OCTAL
-\204\032\151\373\365\315\032\045\064\023\075\343\370\374\270\231
-\320\311\024\267
-END
-CKA_CERT_MD5_HASH MULTILINE_OCTAL
-\270\311\067\337\372\153\061\204\144\305\352\021\152\033\165\374
-END
-CKA_ISSUER MULTILINE_OCTAL
-\060\103\061\013\060\011\006\003\125\004\006\023\002\103\116\061
-\034\060\032\006\003\125\004\012\023\023\151\124\162\165\163\103
-\150\151\156\141\040\103\157\056\054\114\164\144\056\061\026\060
-\024\006\003\125\004\003\023\015\166\124\162\165\163\040\122\157
-\157\164\040\103\101
-END
-CKA_SERIAL_NUMBER MULTILINE_OCTAL
-\002\024\103\343\161\023\330\263\131\024\135\267\316\214\375\065
-\375\157\274\005\215\105
-END
-CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
-CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
-CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
-CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
-
 #
 # Certificate "ISRG Root X2"
 #
@@ -21638,289 +20704,6 @@ CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
 CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
 CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
 
-#
-# Certificate "BJCA Global Root CA1"
-#
-# Issuer: CN=BJCA Global Root CA1,O=BEIJING CERTIFICATE AUTHORITY,C=CN
-# Serial Number:55:6f:65:e3:b4:d9:90:6a:1b:09:d1:6c:3e:c0:6c:20
-# Subject: CN=BJCA Global Root CA1,O=BEIJING CERTIFICATE AUTHORITY,C=CN
-# Not Valid Before: Thu Dec 19 03:16:17 2019
-# Not Valid After : Mon Dec 12 03:16:17 2044
-# Fingerprint (SHA-256): F3:89:6F:88:FE:7C:0A:88:27:66:A7:FA:6A:D2:74:9F:B5:7A:7F:3E:98:FB:76:9C:1F:A7:B0:9C:2C:44:D5:AE
-# Fingerprint (SHA1): D5:EC:8D:7B:4C:BA:79:F4:E7:E8:CB:9D:6B:AE:77:83:10:03:21:6A
-CKA_CLASS CK_OBJECT_CLASS CKO_CERTIFICATE
-CKA_TOKEN CK_BBOOL CK_TRUE
-CKA_PRIVATE CK_BBOOL CK_FALSE
-CKA_MODIFIABLE CK_BBOOL CK_FALSE
-CKA_LABEL UTF8 "BJCA Global Root CA1"
-CKA_CERTIFICATE_TYPE CK_CERTIFICATE_TYPE CKC_X_509
-CKA_SUBJECT MULTILINE_OCTAL
-\060\124\061\013\060\011\006\003\125\004\006\023\002\103\116\061
-\046\060\044\006\003\125\004\012\014\035\102\105\111\112\111\116
-\107\040\103\105\122\124\111\106\111\103\101\124\105\040\101\125
-\124\110\117\122\111\124\131\061\035\060\033\006\003\125\004\003
-\014\024\102\112\103\101\040\107\154\157\142\141\154\040\122\157
-\157\164\040\103\101\061
-END
-CKA_ID UTF8 "0"
-CKA_ISSUER MULTILINE_OCTAL
-\060\124\061\013\060\011\006\003\125\004\006\023\002\103\116\061
-\046\060\044\006\003\125\004\012\014\035\102\105\111\112\111\116
-\107\040\103\105\122\124\111\106\111\103\101\124\105\040\101\125
-\124\110\117\122\111\124\131\061\035\060\033\006\003\125\004\003
-\014\024\102\112\103\101\040\107\154\157\142\141\154\040\122\157
-\157\164\040\103\101\061
-END
-CKA_SERIAL_NUMBER MULTILINE_OCTAL
-\002\020\125\157\145\343\264\331\220\152\033\011\321\154\076\300
-\154\040
-END
-CKA_VALUE MULTILINE_OCTAL
-\060\202\005\164\060\202\003\134\240\003\002\001\002\002\020\125
-\157\145\343\264\331\220\152\033\011\321\154\076\300\154\040\060
-\015\006\011\052\206\110\206\367\015\001\001\013\005\000\060\124
-\061\013\060\011\006\003\125\004\006\023\002\103\116\061\046\060
-\044\006\003\125\004\012\014\035\102\105\111\112\111\116\107\040
-\103\105\122\124\111\106\111\103\101\124\105\040\101\125\124\110
-\117\122\111\124\131\061\035\060\033\006\003\125\004\003\014\024
-\102\112\103\101\040\107\154\157\142\141\154\040\122\157\157\164
-\040\103\101\061\060\036\027\015\061\071\061\062\061\071\060\063
-\061\066\061\067\132\027\015\064\064\061\062\061\062\060\063\061
-\066\061\067\132\060\124\061\013\060\011\006\003\125\004\006\023
-\002\103\116\061\046\060\044\006\003\125\004\012\014\035\102\105
-\111\112\111\116\107\040\103\105\122\124\111\106\111\103\101\124
-\105\040\101\125\124\110\117\122\111\124\131\061\035\060\033\006
-\003\125\004\003\014\024\102\112\103\101\040\107\154\157\142\141
-\154\040\122\157\157\164\040\103\101\061\060\202\002\042\060\015
-\006\011\052\206\110\206\367\015\001\001\001\005\000\003\202\002
-\017\000\060\202\002\012\002\202\002\001\000\361\146\010\275\331
-\305\025\141\313\204\004\101\245\151\067\167\035\301\260\173\372
-\303\167\110\220\023\162\144\321\270\174\220\065\235\030\171\210
-\343\227\001\074\107\201\362\016\242\230\015\236\077\067\340\031
-\262\220\362\106\034\222\261\072\141\316\372\267\106\236\003\206
-\327\063\156\355\367\105\214\166\067\336\156\226\221\367\327\176
-\053\207\027\325\213\065\356\204\221\162\127\334\140\303\303\271
-\347\307\147\044\043\117\143\012\143\366\146\175\113\125\247\077
-\170\144\111\151\022\227\340\114\015\323\011\240\062\060\072\372
-\237\300\362\234\305\022\052\056\034\265\004\063\332\244\070\021
-\152\336\306\030\366\107\072\042\101\207\042\374\304\211\050\124
-\330\214\245\060\012\370\027\026\312\254\067\375\171\247\221\027
-\170\070\231\255\130\355\262\336\314\211\175\003\234\263\211\145
-\347\343\073\261\042\206\217\006\155\170\007\375\221\022\177\260
-\153\034\211\015\371\270\313\164\133\007\302\310\364\065\321\144
-\143\172\351\156\232\050\326\060\275\346\033\335\025\257\204\352
-\234\307\312\365\016\352\362\135\051\207\217\151\163\071\276\056
-\044\157\105\041\254\305\324\151\045\006\203\255\172\110\205\023
-\054\015\006\270\154\171\126\374\243\147\062\201\365\127\245\312
-\127\102\151\351\134\044\141\357\342\060\030\116\104\230\125\157
-\172\302\223\330\031\266\336\174\107\212\021\116\111\107\333\050
-\224\002\013\224\112\054\371\022\320\117\350\061\176\154\172\277
-\246\077\233\071\075\002\026\243\030\263\147\254\133\077\054\203
-\053\147\071\201\134\271\176\224\325\144\335\236\217\156\256\350
-\174\133\264\327\152\107\110\327\176\263\324\055\216\126\166\116
-\317\151\361\156\104\154\324\044\352\215\044\241\030\277\275\127
-\376\251\231\065\265\333\020\167\270\075\110\272\326\301\347\361
-\043\076\327\337\205\235\047\074\324\100\275\012\014\275\365\347
-\215\045\326\201\164\207\106\324\051\165\242\102\154\367\163\211
-\347\175\277\172\112\037\323\042\311\025\125\317\337\157\174\125
-\320\244\213\007\021\067\137\203\246\046\127\246\001\133\176\376
-\130\150\007\251\351\172\331\271\350\377\120\037\253\302\264\300
-\316\350\352\375\017\275\215\115\270\274\161\002\003\001\000\001
-\243\102\060\100\060\035\006\003\125\035\016\004\026\004\024\305
-\357\355\314\330\215\041\306\110\344\343\327\024\056\247\026\223
-\345\230\001\060\017\006\003\125\035\023\001\001\377\004\005\060
-\003\001\001\377\060\016\006\003\125\035\017\001\001\377\004\004
-\003\002\001\006\060\015\006\011\052\206\110\206\367\015\001\001
-\013\005\000\003\202\002\001\000\122\202\254\041\064\037\043\362
-\242\330\371\270\257\067\066\040\211\321\067\003\326\151\237\270
-\141\020\272\242\061\230\131\107\350\321\015\045\036\025\101\014
-\340\052\125\325\127\122\313\370\344\307\151\243\035\115\161\002
-\136\137\041\105\140\110\134\011\216\111\020\301\004\334\251\142
-\153\002\360\103\310\116\235\070\111\164\311\062\160\124\155\301
-\107\374\216\264\066\236\324\234\275\335\040\326\123\311\030\251
-\265\126\271\166\213\225\147\146\356\275\230\376\256\357\276\156
-\373\140\366\375\131\306\052\033\077\043\112\224\044\060\047\310
-\211\274\353\104\044\232\313\075\276\117\325\172\316\216\027\313
-\142\301\331\336\036\016\172\377\103\206\064\122\274\141\077\074
-\137\273\331\166\264\123\274\227\263\376\212\114\022\056\053\363
-\327\316\341\242\377\335\173\160\373\073\241\115\244\143\002\375
-\070\227\225\077\005\160\240\153\337\142\201\103\213\264\131\015
-\112\214\124\234\305\273\201\237\315\175\245\357\013\045\036\072
-\040\333\034\374\037\230\147\002\012\324\163\104\023\333\121\204
-\032\125\003\126\340\000\176\164\006\377\070\304\162\035\323\250
-\077\150\061\135\323\011\307\056\214\133\143\340\350\334\036\322
-\354\141\036\362\336\345\357\366\231\166\140\055\036\224\162\161
-\306\013\052\062\307\222\116\325\106\327\035\371\251\031\012\310
-\372\225\316\155\043\230\252\013\070\255\232\126\015\157\215\361
-\061\000\210\301\027\234\315\031\066\065\376\125\123\240\340\074
-\063\137\226\136\342\062\351\337\063\273\006\112\251\330\204\163
-\316\167\322\306\254\161\341\134\243\035\014\273\012\337\137\342
-\243\161\330\332\067\132\240\170\053\364\324\175\353\166\355\362
-\141\160\245\145\232\323\211\064\030\253\373\162\076\327\264\075
-\171\134\330\037\241\063\173\331\202\120\014\223\027\252\154\334
-\302\202\273\002\127\066\257\230\047\052\071\120\341\260\211\365
-\045\227\176\107\150\020\264\354\163\312\263\227\321\044\334\366
-\142\240\050\323\265\243\270\144\267\210\142\102\317\235\123\315
-\231\276\144\150\217\117\036\022\110\367\322\051\303\230\050\312
-\362\062\013\223\214\051\117\074\140\062\315\005\226\141\354\362
-\257\376\263\160\054\056\246\362
-END
-CKA_NSS_MOZILLA_CA_POLICY CK_BBOOL CK_TRUE
-CKA_NSS_SERVER_DISTRUST_AFTER CK_BBOOL CK_FALSE
-CKA_NSS_EMAIL_DISTRUST_AFTER CK_BBOOL CK_FALSE
-
-# Trust for "BJCA Global Root CA1"
-# Issuer: CN=BJCA Global Root CA1,O=BEIJING CERTIFICATE AUTHORITY,C=CN
-# Serial Number:55:6f:65:e3:b4:d9:90:6a:1b:09:d1:6c:3e:c0:6c:20
-# Subject: CN=BJCA Global Root CA1,O=BEIJING CERTIFICATE AUTHORITY,C=CN
-# Not Valid Before: Thu Dec 19 03:16:17 2019
-# Not Valid After : Mon Dec 12 03:16:17 2044
-# Fingerprint (SHA-256): F3:89:6F:88:FE:7C:0A:88:27:66:A7:FA:6A:D2:74:9F:B5:7A:7F:3E:98:FB:76:9C:1F:A7:B0:9C:2C:44:D5:AE
-# Fingerprint (SHA1): D5:EC:8D:7B:4C:BA:79:F4:E7:E8:CB:9D:6B:AE:77:83:10:03:21:6A
-CKA_CLASS CK_OBJECT_CLASS CKO_NSS_TRUST
-CKA_TOKEN CK_BBOOL CK_TRUE
-CKA_PRIVATE CK_BBOOL CK_FALSE
-CKA_MODIFIABLE CK_BBOOL CK_FALSE
-CKA_LABEL UTF8 "BJCA Global Root CA1"
-CKA_CERT_SHA1_HASH MULTILINE_OCTAL
-\325\354\215\173\114\272\171\364\347\350\313\235\153\256\167\203
-\020\003\041\152
-END
-CKA_CERT_MD5_HASH MULTILINE_OCTAL
-\102\062\231\166\103\063\066\044\065\007\202\233\050\371\320\220
-END
-CKA_ISSUER MULTILINE_OCTAL
-\060\124\061\013\060\011\006\003\125\004\006\023\002\103\116\061
-\046\060\044\006\003\125\004\012\014\035\102\105\111\112\111\116
-\107\040\103\105\122\124\111\106\111\103\101\124\105\040\101\125
-\124\110\117\122\111\124\131\061\035\060\033\006\003\125\004\003
-\014\024\102\112\103\101\040\107\154\157\142\141\154\040\122\157
-\157\164\040\103\101\061
-END
-CKA_SERIAL_NUMBER MULTILINE_OCTAL
-\002\020\125\157\145\343\264\331\220\152\033\011\321\154\076\300
-\154\040
-END
-CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
-CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
-CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
-CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
-
-#
-# Certificate "BJCA Global Root CA2"
-#
-# Issuer: CN=BJCA Global Root CA2,O=BEIJING CERTIFICATE AUTHORITY,C=CN
-# Serial Number:2c:17:08:7d:64:2a:c0:fe:85:18:59:06:cf:b4:4a:eb
-# Subject: CN=BJCA Global Root CA2,O=BEIJING CERTIFICATE AUTHORITY,C=CN
-# Not Valid Before: Thu Dec 19 03:18:21 2019
-# Not Valid After : Mon Dec 12 03:18:21 2044
-# Fingerprint (SHA-256): 57:4D:F6:93:1E:27:80:39:66:7B:72:0A:FD:C1:60:0F:C2:7E:B6:6D:D3:09:29:79:FB:73:85:64:87:21:28:82
-# Fingerprint (SHA1): F4:27:86:EB:6E:B8:6D:88:31:67:02:FB:BA:66:A4:53:00:AA:7A:A6
-CKA_CLASS CK_OBJECT_CLASS CKO_CERTIFICATE
-CKA_TOKEN CK_BBOOL CK_TRUE
-CKA_PRIVATE CK_BBOOL CK_FALSE
-CKA_MODIFIABLE CK_BBOOL CK_FALSE
-CKA_LABEL UTF8 "BJCA Global Root CA2"
-CKA_CERTIFICATE_TYPE CK_CERTIFICATE_TYPE CKC_X_509
-CKA_SUBJECT MULTILINE_OCTAL
-\060\124\061\013\060\011\006\003\125\004\006\023\002\103\116\061
-\046\060\044\006\003\125\004\012\014\035\102\105\111\112\111\116
-\107\040\103\105\122\124\111\106\111\103\101\124\105\040\101\125
-\124\110\117\122\111\124\131\061\035\060\033\006\003\125\004\003
-\014\024\102\112\103\101\040\107\154\157\142\141\154\040\122\157
-\157\164\040\103\101\062
-END
-CKA_ID UTF8 "0"
-CKA_ISSUER MULTILINE_OCTAL
-\060\124\061\013\060\011\006\003\125\004\006\023\002\103\116\061
-\046\060\044\006\003\125\004\012\014\035\102\105\111\112\111\116
-\107\040\103\105\122\124\111\106\111\103\101\124\105\040\101\125
-\124\110\117\122\111\124\131\061\035\060\033\006\003\125\004\003
-\014\024\102\112\103\101\040\107\154\157\142\141\154\040\122\157
-\157\164\040\103\101\062
-END
-CKA_SERIAL_NUMBER MULTILINE_OCTAL
-\002\020\054\027\010\175\144\052\300\376\205\030\131\006\317\264
-\112\353
-END
-CKA_VALUE MULTILINE_OCTAL
-\060\202\002\045\060\202\001\253\240\003\002\001\002\002\020\054
-\027\010\175\144\052\300\376\205\030\131\006\317\264\112\353\060
-\012\006\010\052\206\110\316\075\004\003\003\060\124\061\013\060
-\011\006\003\125\004\006\023\002\103\116\061\046\060\044\006\003
-\125\004\012\014\035\102\105\111\112\111\116\107\040\103\105\122
-\124\111\106\111\103\101\124\105\040\101\125\124\110\117\122\111
-\124\131\061\035\060\033\006\003\125\004\003\014\024\102\112\103
-\101\040\107\154\157\142\141\154\040\122\157\157\164\040\103\101
-\062\060\036\027\015\061\071\061\062\061\071\060\063\061\070\062
-\061\132\027\015\064\064\061\062\061\062\060\063\061\070\062\061
-\132\060\124\061\013\060\011\006\003\125\004\006\023\002\103\116
-\061\046\060\044\006\003\125\004\012\014\035\102\105\111\112\111
-\116\107\040\103\105\122\124\111\106\111\103\101\124\105\040\101
-\125\124\110\117\122\111\124\131\061\035\060\033\006\003\125\004
-\003\014\024\102\112\103\101\040\107\154\157\142\141\154\040\122
-\157\157\164\040\103\101\062\060\166\060\020\006\007\052\206\110
-\316\075\002\001\006\005\053\201\004\000\042\003\142\000\004\235
-\313\200\221\215\123\147\265\271\120\261\003\370\345\111\037\101
-\042\011\260\121\122\130\326\053\064\217\305\022\106\024\305\213
-\057\054\204\377\054\156\250\325\361\011\343\003\041\024\304\103
-\075\174\301\054\304\113\152\112\315\351\207\340\175\366\042\276
-\372\112\121\270\060\212\375\341\336\030\022\012\366\107\267\347
-\027\277\047\212\324\101\114\226\074\140\226\301\375\025\034\243
-\102\060\100\060\035\006\003\125\035\016\004\026\004\024\322\112
-\261\121\177\006\360\321\202\037\116\156\137\253\203\374\110\324
-\260\221\060\017\006\003\125\035\023\001\001\377\004\005\060\003
-\001\001\377\060\016\006\003\125\035\017\001\001\377\004\004\003
-\002\001\006\060\012\006\010\052\206\110\316\075\004\003\003\003
-\150\000\060\145\002\060\032\274\133\327\376\251\322\124\016\112
-\135\322\155\261\100\334\364\103\325\322\112\231\031\022\126\200
-\367\203\064\341\065\116\110\155\004\017\127\061\060\060\055\261
-\252\235\003\070\333\006\002\061\000\313\314\207\123\313\172\337
-\040\121\163\220\300\250\133\141\320\305\120\071\375\205\376\301
-\343\170\370\246\326\113\275\233\207\217\017\345\326\123\226\253
-\074\310\100\332\141\367\123\243\367
-END
-CKA_NSS_MOZILLA_CA_POLICY CK_BBOOL CK_TRUE
-CKA_NSS_SERVER_DISTRUST_AFTER CK_BBOOL CK_FALSE
-CKA_NSS_EMAIL_DISTRUST_AFTER CK_BBOOL CK_FALSE
-
-# Trust for "BJCA Global Root CA2"
-# Issuer: CN=BJCA Global Root CA2,O=BEIJING CERTIFICATE AUTHORITY,C=CN
-# Serial Number:2c:17:08:7d:64:2a:c0:fe:85:18:59:06:cf:b4:4a:eb
-# Subject: CN=BJCA Global Root CA2,O=BEIJING CERTIFICATE AUTHORITY,C=CN
-# Not Valid Before: Thu Dec 19 03:18:21 2019
-# Not Valid After : Mon Dec 12 03:18:21 2044
-# Fingerprint (SHA-256): 57:4D:F6:93:1E:27:80:39:66:7B:72:0A:FD:C1:60:0F:C2:7E:B6:6D:D3:09:29:79:FB:73:85:64:87:21:28:82
-# Fingerprint (SHA1): F4:27:86:EB:6E:B8:6D:88:31:67:02:FB:BA:66:A4:53:00:AA:7A:A6
-CKA_CLASS CK_OBJECT_CLASS CKO_NSS_TRUST
-CKA_TOKEN CK_BBOOL CK_TRUE
-CKA_PRIVATE CK_BBOOL CK_FALSE
-CKA_MODIFIABLE CK_BBOOL CK_FALSE
-CKA_LABEL UTF8 "BJCA Global Root CA2"
-CKA_CERT_SHA1_HASH MULTILINE_OCTAL
-\364\047\206\353\156\270\155\210\061\147\002\373\272\146\244\123
-\000\252\172\246
-END
-CKA_CERT_MD5_HASH MULTILINE_OCTAL
-\136\012\366\107\137\246\024\350\021\001\225\077\115\001\353\074
-END
-CKA_ISSUER MULTILINE_OCTAL
-\060\124\061\013\060\011\006\003\125\004\006\023\002\103\116\061
-\046\060\044\006\003\125\004\012\014\035\102\105\111\112\111\116
-\107\040\103\105\122\124\111\106\111\103\101\124\105\040\101\125
-\124\110\117\122\111\124\131\061\035\060\033\006\003\125\004\003
-\014\024\102\112\103\101\040\107\154\157\142\141\154\040\122\157
-\157\164\040\103\101\062
-END
-CKA_SERIAL_NUMBER MULTILINE_OCTAL
-\002\020\054\027\010\175\144\052\300\376\205\030\131\006\317\264
-\112\353
-END
-CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
-CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
-CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
-CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
-
 #
 # Certificate "LAWtrust Root CA2 (4096)"
 #
@@ -23778,295 +22561,6 @@ CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
 CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
 CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
 
-#
-# Certificate "TrustAsia Global Root CA G3"
-#
-# Issuer: CN=TrustAsia Global Root CA G3,O="TrustAsia Technologies, Inc.",C=CN
-# Serial Number:64:f6:0e:65:77:61:6a:ab:3b:b4:ea:85:84:bb:b1:89:b8:71:93:0f
-# Subject: CN=TrustAsia Global Root CA G3,O="TrustAsia Technologies, Inc.",C=CN
-# Not Valid Before: Thu May 20 02:10:19 2021
-# Not Valid After : Sat May 19 02:10:19 2046
-# Fingerprint (SHA-256): E0:D3:22:6A:EB:11:63:C2:E4:8F:F9:BE:3B:50:B4:C6:43:1B:E7:BB:1E:AC:C5:C3:6B:5D:5E:C5:09:03:9A:08
-# Fingerprint (SHA1): 63:CF:B6:C1:27:2B:56:E4:88:8E:1C:23:9A:B6:2E:81:47:24:C3:C7
-CKA_CLASS CK_OBJECT_CLASS CKO_CERTIFICATE
-CKA_TOKEN CK_BBOOL CK_TRUE
-CKA_PRIVATE CK_BBOOL CK_FALSE
-CKA_MODIFIABLE CK_BBOOL CK_FALSE
-CKA_LABEL UTF8 "TrustAsia Global Root CA G3"
-CKA_CERTIFICATE_TYPE CK_CERTIFICATE_TYPE CKC_X_509
-CKA_SUBJECT MULTILINE_OCTAL
-\060\132\061\013\060\011\006\003\125\004\006\023\002\103\116\061
-\045\060\043\006\003\125\004\012\014\034\124\162\165\163\164\101
-\163\151\141\040\124\145\143\150\156\157\154\157\147\151\145\163
-\054\040\111\156\143\056\061\044\060\042\006\003\125\004\003\014
-\033\124\162\165\163\164\101\163\151\141\040\107\154\157\142\141
-\154\040\122\157\157\164\040\103\101\040\107\063
-END
-CKA_ID UTF8 "0"
-CKA_ISSUER MULTILINE_OCTAL
-\060\132\061\013\060\011\006\003\125\004\006\023\002\103\116\061
-\045\060\043\006\003\125\004\012\014\034\124\162\165\163\164\101
-\163\151\141\040\124\145\143\150\156\157\154\157\147\151\145\163
-\054\040\111\156\143\056\061\044\060\042\006\003\125\004\003\014
-\033\124\162\165\163\164\101\163\151\141\040\107\154\157\142\141
-\154\040\122\157\157\164\040\103\101\040\107\063
-END
-CKA_SERIAL_NUMBER MULTILINE_OCTAL
-\002\024\144\366\016\145\167\141\152\253\073\264\352\205\204\273
-\261\211\270\161\223\017
-END
-CKA_VALUE MULTILINE_OCTAL
-\060\202\005\245\060\202\003\215\240\003\002\001\002\002\024\144
-\366\016\145\167\141\152\253\073\264\352\205\204\273\261\211\270
-\161\223\017\060\015\006\011\052\206\110\206\367\015\001\001\014
-\005\000\060\132\061\013\060\011\006\003\125\004\006\023\002\103
-\116\061\045\060\043\006\003\125\004\012\014\034\124\162\165\163
-\164\101\163\151\141\040\124\145\143\150\156\157\154\157\147\151
-\145\163\054\040\111\156\143\056\061\044\060\042\006\003\125\004
-\003\014\033\124\162\165\163\164\101\163\151\141\040\107\154\157
-\142\141\154\040\122\157\157\164\040\103\101\040\107\063\060\036
-\027\015\062\061\060\065\062\060\060\062\061\060\061\071\132\027
-\015\064\066\060\065\061\071\060\062\061\060\061\071\132\060\132
-\061\013\060\011\006\003\125\004\006\023\002\103\116\061\045\060
-\043\006\003\125\004\012\014\034\124\162\165\163\164\101\163\151
-\141\040\124\145\143\150\156\157\154\157\147\151\145\163\054\040
-\111\156\143\056\061\044\060\042\006\003\125\004\003\014\033\124
-\162\165\163\164\101\163\151\141\040\107\154\157\142\141\154\040
-\122\157\157\164\040\103\101\040\107\063\060\202\002\042\060\015
-\006\011\052\206\110\206\367\015\001\001\001\005\000\003\202\002
-\017\000\060\202\002\012\002\202\002\001\000\300\061\202\141\222
-\344\224\033\012\052\145\320\276\006\251\207\073\121\022\352\160
-\101\256\342\373\164\352\012\215\271\263\114\334\217\267\023\122
-\117\124\030\341\054\163\225\221\305\146\073\152\317\254\143\155
-\207\123\360\367\361\071\267\240\103\143\260\304\003\135\127\251
-\347\104\316\304\241\203\145\366\120\076\261\176\026\270\072\212
-\002\320\226\037\000\315\005\041\357\006\155\335\041\234\031\103
-\105\241\305\350\200\312\302\255\100\142\027\006\306\252\274\363
-\326\346\374\120\176\146\102\037\074\213\246\171\171\206\100\065
-\237\040\357\077\353\213\107\037\217\216\305\324\216\266\054\311
-\104\004\343\324\103\165\077\325\077\257\034\314\176\106\137\254
-\337\144\020\212\357\106\360\220\360\017\055\364\210\013\261\051
-\252\257\205\252\111\130\250\277\143\240\070\221\346\263\346\167
-\150\304\371\052\031\204\273\016\341\365\257\211\354\245\057\120
-\040\164\036\022\101\163\036\044\331\312\316\054\241\131\065\300
-\310\035\106\047\141\132\217\371\115\323\162\171\146\036\237\025
-\220\041\055\375\355\213\126\160\003\112\111\076\177\151\061\022
-\151\307\036\134\312\172\023\213\350\346\365\140\017\314\223\054
-\204\177\361\374\152\374\233\107\235\333\255\210\075\363\166\165
-\063\327\113\244\310\213\371\365\103\130\117\313\310\003\124\217
-\245\205\170\004\032\363\163\362\327\207\035\101\237\347\330\027
-\316\032\234\017\112\374\334\104\150\124\150\342\101\074\376\054
-\204\206\067\074\315\077\057\242\333\347\367\124\003\137\131\323
-\367\221\170\307\213\167\152\026\345\111\205\220\105\162\160\057
-\221\135\370\076\145\100\013\031\231\311\046\040\132\150\301\065
-\277\117\247\121\361\330\021\053\133\340\232\236\050\073\012\072
-\012\037\301\201\345\056\360\246\271\151\245\210\224\346\153\023
-\177\321\144\077\075\234\160\106\345\242\205\173\130\204\047\334
-\304\200\076\147\232\232\307\232\061\016\060\354\346\027\100\225
-\331\105\355\001\226\252\277\014\363\113\321\143\367\023\130\300
-\270\363\372\147\335\233\175\155\112\377\062\114\265\045\073\377
-\034\147\017\205\042\131\005\221\221\101\167\201\320\205\114\207
-\020\161\377\236\103\033\256\225\165\055\201\002\003\001\000\001
-\243\143\060\141\060\017\006\003\125\035\023\001\001\377\004\005
-\060\003\001\001\377\060\037\006\003\125\035\043\004\030\060\026
-\200\024\100\344\344\362\043\357\070\312\260\256\127\177\362\041
-\060\026\064\333\274\222\060\035\006\003\125\035\016\004\026\004
-\024\100\344\344\362\043\357\070\312\260\256\127\177\362\041\060
-\026\064\333\274\222\060\016\006\003\125\035\017\001\001\377\004
-\004\003\002\001\006\060\015\006\011\052\206\110\206\367\015\001
-\001\014\005\000\003\202\002\001\000\046\073\121\341\115\070\363
-\062\030\264\264\136\341\145\136\304\224\117\324\247\141\243\370
-\300\317\063\001\002\351\303\252\065\017\361\224\023\167\167\065
-\236\055\126\121\104\156\341\306\056\050\036\377\332\354\107\315
-\227\104\027\367\340\114\302\341\174\174\062\172\146\310\132\266
-\134\123\105\127\132\105\324\005\231\057\056\043\125\356\143\150
-\337\323\033\170\247\022\224\006\000\165\015\162\204\351\056\274
-\132\152\325\336\057\131\307\243\354\322\207\146\333\267\124\265
-\044\253\364\103\170\333\113\004\304\157\335\346\076\146\076\051
-\362\113\150\161\042\207\240\370\261\063\143\166\343\015\205\162
-\104\042\125\077\034\174\351\374\270\025\350\122\372\252\076\243
-\041\071\065\164\211\246\152\302\071\372\170\317\266\254\347\347
-\326\126\377\043\222\056\120\013\251\265\007\063\364\070\137\244
-\111\246\313\145\160\166\350\012\205\200\113\066\075\063\367\225
-\124\165\045\332\254\304\163\202\145\351\122\365\134\375\070\225
-\002\152\151\060\305\034\012\127\007\256\042\244\054\371\305\101
-\267\270\354\237\117\110\000\371\001\004\125\314\254\371\062\061
-\304\165\225\006\240\177\321\215\047\335\263\251\244\162\207\376
-\131\213\232\172\164\026\335\026\245\142\051\353\072\226\334\213
-\247\150\131\323\353\167\221\071\370\327\313\331\217\137\132\047
-\001\175\135\150\031\142\330\310\315\364\267\162\107\276\133\227
-\316\362\255\242\231\223\255\224\313\223\366\022\011\225\266\253
-\327\073\320\077\021\313\060\026\056\171\200\344\147\201\055\135
-\355\160\170\266\140\131\254\341\135\105\143\217\310\337\162\150
-\133\352\035\270\001\361\176\373\347\212\263\343\124\240\070\011
-\340\074\336\102\362\302\355\056\233\363\037\065\266\066\330\343
-\200\241\213\315\231\144\017\302\252\253\261\312\365\157\236\103
-\215\204\124\231\263\156\300\022\146\330\160\020\361\006\065\063
-\103\250\234\056\272\024\061\316\020\177\034\206\343\217\322\325
-\370\167\354\233\253\361\057\143\331\102\137\340\147\201\144\221
-\361\227\057\374\156\046\366\063\370\323\265\370\304\142\253\061
-\121\045\002\172\370\335\153\145\325\155\115\060\310\145\272\150
-\024\145\254\047\013\164\212\362\207
-END
-CKA_NSS_MOZILLA_CA_POLICY CK_BBOOL CK_TRUE
-CKA_NSS_SERVER_DISTRUST_AFTER CK_BBOOL CK_FALSE
-CKA_NSS_EMAIL_DISTRUST_AFTER CK_BBOOL CK_FALSE
-
-# Trust for "TrustAsia Global Root CA G3"
-# Issuer: CN=TrustAsia Global Root CA G3,O="TrustAsia Technologies, Inc.",C=CN
-# Serial Number:64:f6:0e:65:77:61:6a:ab:3b:b4:ea:85:84:bb:b1:89:b8:71:93:0f
-# Subject: CN=TrustAsia Global Root CA G3,O="TrustAsia Technologies, Inc.",C=CN
-# Not Valid Before: Thu May 20 02:10:19 2021
-# Not Valid After : Sat May 19 02:10:19 2046
-# Fingerprint (SHA-256): E0:D3:22:6A:EB:11:63:C2:E4:8F:F9:BE:3B:50:B4:C6:43:1B:E7:BB:1E:AC:C5:C3:6B:5D:5E:C5:09:03:9A:08
-# Fingerprint (SHA1): 63:CF:B6:C1:27:2B:56:E4:88:8E:1C:23:9A:B6:2E:81:47:24:C3:C7
-CKA_CLASS CK_OBJECT_CLASS CKO_NSS_TRUST
-CKA_TOKEN CK_BBOOL CK_TRUE
-CKA_PRIVATE CK_BBOOL CK_FALSE
-CKA_MODIFIABLE CK_BBOOL CK_FALSE
-CKA_LABEL UTF8 "TrustAsia Global Root CA G3"
-CKA_CERT_SHA1_HASH MULTILINE_OCTAL
-\143\317\266\301\047\053\126\344\210\216\034\043\232\266\056\201
-\107\044\303\307
-END
-CKA_CERT_MD5_HASH MULTILINE_OCTAL
-\060\102\033\267\273\201\165\065\344\026\117\123\322\224\336\004
-END
-CKA_ISSUER MULTILINE_OCTAL
-\060\132\061\013\060\011\006\003\125\004\006\023\002\103\116\061
-\045\060\043\006\003\125\004\012\014\034\124\162\165\163\164\101
-\163\151\141\040\124\145\143\150\156\157\154\157\147\151\145\163
-\054\040\111\156\143\056\061\044\060\042\006\003\125\004\003\014
-\033\124\162\165\163\164\101\163\151\141\040\107\154\157\142\141
-\154\040\122\157\157\164\040\103\101\040\107\063
-END
-CKA_SERIAL_NUMBER MULTILINE_OCTAL
-\002\024\144\366\016\145\167\141\152\253\073\264\352\205\204\273
-\261\211\270\161\223\017
-END
-CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
-CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
-CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
-CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
-
-#
-# Certificate "TrustAsia Global Root CA G4"
-#
-# Issuer: CN=TrustAsia Global Root CA G4,O="TrustAsia Technologies, Inc.",C=CN
-# Serial Number:4f:23:64:b8:8e:97:63:9e:c6:53:81:c1:76:4e:cb:2a:74:15:d6:d7
-# Subject: CN=TrustAsia Global Root CA G4,O="TrustAsia Technologies, Inc.",C=CN
-# Not Valid Before: Thu May 20 02:10:22 2021
-# Not Valid After : Sat May 19 02:10:22 2046
-# Fingerprint (SHA-256): BE:4B:56:CB:50:56:C0:13:6A:52:6D:F4:44:50:8D:AA:36:A0:B5:4F:42:E4:AC:38:F7:2A:F4:70:E4:79:65:4C
-# Fingerprint (SHA1): 57:73:A5:61:5D:80:B2:E6:AC:38:82:FC:68:07:31:AC:9F:B5:92:5A
-CKA_CLASS CK_OBJECT_CLASS CKO_CERTIFICATE
-CKA_TOKEN CK_BBOOL CK_TRUE
-CKA_PRIVATE CK_BBOOL CK_FALSE
-CKA_MODIFIABLE CK_BBOOL CK_FALSE
-CKA_LABEL UTF8 "TrustAsia Global Root CA G4"
-CKA_CERTIFICATE_TYPE CK_CERTIFICATE_TYPE CKC_X_509
-CKA_SUBJECT MULTILINE_OCTAL
-\060\132\061\013\060\011\006\003\125\004\006\023\002\103\116\061
-\045\060\043\006\003\125\004\012\014\034\124\162\165\163\164\101
-\163\151\141\040\124\145\143\150\156\157\154\157\147\151\145\163
-\054\040\111\156\143\056\061\044\060\042\006\003\125\004\003\014
-\033\124\162\165\163\164\101\163\151\141\040\107\154\157\142\141
-\154\040\122\157\157\164\040\103\101\040\107\064
-END
-CKA_ID UTF8 "0"
-CKA_ISSUER MULTILINE_OCTAL
-\060\132\061\013\060\011\006\003\125\004\006\023\002\103\116\061
-\045\060\043\006\003\125\004\012\014\034\124\162\165\163\164\101
-\163\151\141\040\124\145\143\150\156\157\154\157\147\151\145\163
-\054\040\111\156\143\056\061\044\060\042\006\003\125\004\003\014
-\033\124\162\165\163\164\101\163\151\141\040\107\154\157\142\141
-\154\040\122\157\157\164\040\103\101\040\107\064
-END
-CKA_SERIAL_NUMBER MULTILINE_OCTAL
-\002\024\117\043\144\270\216\227\143\236\306\123\201\301\166\116
-\313\052\164\025\326\327
-END
-CKA_VALUE MULTILINE_OCTAL
-\060\202\002\125\060\202\001\334\240\003\002\001\002\002\024\117
-\043\144\270\216\227\143\236\306\123\201\301\166\116\313\052\164
-\025\326\327\060\012\006\010\052\206\110\316\075\004\003\003\060
-\132\061\013\060\011\006\003\125\004\006\023\002\103\116\061\045
-\060\043\006\003\125\004\012\014\034\124\162\165\163\164\101\163
-\151\141\040\124\145\143\150\156\157\154\157\147\151\145\163\054
-\040\111\156\143\056\061\044\060\042\006\003\125\004\003\014\033
-\124\162\165\163\164\101\163\151\141\040\107\154\157\142\141\154
-\040\122\157\157\164\040\103\101\040\107\064\060\036\027\015\062
-\061\060\065\062\060\060\062\061\060\062\062\132\027\015\064\066
-\060\065\061\071\060\062\061\060\062\062\132\060\132\061\013\060
-\011\006\003\125\004\006\023\002\103\116\061\045\060\043\006\003
-\125\004\012\014\034\124\162\165\163\164\101\163\151\141\040\124
-\145\143\150\156\157\154\157\147\151\145\163\054\040\111\156\143
-\056\061\044\060\042\006\003\125\004\003\014\033\124\162\165\163
-\164\101\163\151\141\040\107\154\157\142\141\154\040\122\157\157
-\164\040\103\101\040\107\064\060\166\060\020\006\007\052\206\110
-\316\075\002\001\006\005\053\201\004\000\042\003\142\000\004\361
-\263\315\070\344\045\103\345\336\031\011\273\201\171\242\025\137
-\025\143\001\336\302\253\335\263\246\033\147\113\200\203\257\231
-\313\254\027\333\053\226\312\174\122\125\342\032\341\075\126\360
-\057\026\010\372\025\274\233\273\107\346\077\356\250\341\114\214
-\365\323\066\371\070\135\253\160\232\107\015\342\201\101\006\353
-\111\371\260\051\335\063\354\120\245\177\171\051\270\040\230\243
-\143\060\141\060\017\006\003\125\035\023\001\001\377\004\005\060
-\003\001\001\377\060\037\006\003\125\035\043\004\030\060\026\200
-\024\245\273\112\227\316\263\053\177\244\061\336\227\203\131\203
-\246\157\161\313\336\060\035\006\003\125\035\016\004\026\004\024
-\245\273\112\227\316\263\053\177\244\061\336\227\203\131\203\246
-\157\161\313\336\060\016\006\003\125\035\017\001\001\377\004\004
-\003\002\001\006\060\012\006\010\052\206\110\316\075\004\003\003
-\003\147\000\060\144\002\060\136\362\353\006\314\111\061\237\100
-\000\155\267\176\066\360\115\021\117\363\313\211\072\054\170\221
-\120\243\133\300\312\165\046\362\277\220\135\013\202\214\140\050
-\237\306\160\232\150\344\361\002\060\134\130\016\126\166\317\130
-\303\327\020\214\272\216\256\343\274\144\165\107\305\125\220\343
-\375\272\125\353\007\304\123\253\067\251\356\041\262\041\133\140
-\217\075\062\361\325\043\224\326\130
-END
-CKA_NSS_MOZILLA_CA_POLICY CK_BBOOL CK_TRUE
-CKA_NSS_SERVER_DISTRUST_AFTER CK_BBOOL CK_FALSE
-CKA_NSS_EMAIL_DISTRUST_AFTER CK_BBOOL CK_FALSE
-
-# Trust for "TrustAsia Global Root CA G4"
-# Issuer: CN=TrustAsia Global Root CA G4,O="TrustAsia Technologies, Inc.",C=CN
-# Serial Number:4f:23:64:b8:8e:97:63:9e:c6:53:81:c1:76:4e:cb:2a:74:15:d6:d7
-# Subject: CN=TrustAsia Global Root CA G4,O="TrustAsia Technologies, Inc.",C=CN
-# Not Valid Before: Thu May 20 02:10:22 2021
-# Not Valid After : Sat May 19 02:10:22 2046
-# Fingerprint (SHA-256): BE:4B:56:CB:50:56:C0:13:6A:52:6D:F4:44:50:8D:AA:36:A0:B5:4F:42:E4:AC:38:F7:2A:F4:70:E4:79:65:4C
-# Fingerprint (SHA1): 57:73:A5:61:5D:80:B2:E6:AC:38:82:FC:68:07:31:AC:9F:B5:92:5A
-CKA_CLASS CK_OBJECT_CLASS CKO_NSS_TRUST
-CKA_TOKEN CK_BBOOL CK_TRUE
-CKA_PRIVATE CK_BBOOL CK_FALSE
-CKA_MODIFIABLE CK_BBOOL CK_FALSE
-CKA_LABEL UTF8 "TrustAsia Global Root CA G4"
-CKA_CERT_SHA1_HASH MULTILINE_OCTAL
-\127\163\245\141\135\200\262\346\254\070\202\374\150\007\061\254
-\237\265\222\132
-END
-CKA_CERT_MD5_HASH MULTILINE_OCTAL
-\124\335\262\327\137\330\076\355\174\340\013\056\314\355\353\353
-END
-CKA_ISSUER MULTILINE_OCTAL
-\060\132\061\013\060\011\006\003\125\004\006\023\002\103\116\061
-\045\060\043\006\003\125\004\012\014\034\124\162\165\163\164\101
-\163\151\141\040\124\145\143\150\156\157\154\157\147\151\145\163
-\054\040\111\156\143\056\061\044\060\042\006\003\125\004\003\014
-\033\124\162\165\163\164\101\163\151\141\040\107\154\157\142\141
-\154\040\122\157\157\164\040\103\101\040\107\064
-END
-CKA_SERIAL_NUMBER MULTILINE_OCTAL
-\002\024\117\043\144\270\216\227\143\236\306\123\201\301\166\116
-\313\052\164\025\326\327
-END
-CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
-CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
-CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
-CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
-
 #
 # Certificate "CommScope Public Trust ECC Root-01"
 #
